<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8" />
  <meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes" />
  <title>Reffy report</title>
    <link rel="stylesheet" href="https://tidoust.github.io/reffy-reports/common/font-awesome.css" />
  <link rel="stylesheet" href="https://unpkg.com/purecss@1.0.0/build/pure-min.css" integrity="sha384-nn4HPE8lTHyVtfCBi5yW9d20FjT8BJwUXyWZT9InLYax14RDjBj46LmSztkmNP9w" crossorigin="anonymous" />
  <link rel="stylesheet" href="https://tidoust.github.io/reffy-reports/common/styles.css" />
  <!--[if lte IE 8]>
      <link rel="stylesheet" href="https://unpkg.com/purecss@1.0.0/build/grids-responsive-old-ie-min.css" />
  <![endif]-->
  <!--[if gt IE 8]><!-->
      <link rel="stylesheet" href="https://unpkg.com/purecss@1.0.0/build/grids-responsive-min.css" />
  <!--<![endif]-->
    <!--[if lt IE 9]>
    <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.3/html5shiv-printshiv.min.js"></script>
  <![endif]-->
  </head>
<body>
    <h1>Reffy crawl report (per issue)</h1>
  <p><a href="https://github.com/tidoust/reffy">Reffy</a> is a <i>spec exploration tool</i>. It crawls a list of specifications to study the WebIDL content, the links that they contain, and the references that these specifications contain.</p>
  <p>113 specifications were crawled in this report.</p>
  <section id="specifications-that-could-not-be-rendered" class="level2">
  <h2>Specifications that could not be rendered</h2>
  <p>Reffy could not fetch or render these specifications for some reason. This may happen when a network error occurred or when a specification uses an old version of ReSpec.</p>
  <ul>
  <li><a href="https://www.w3.org/TR/2dcontext">2dcontext</a>: <code>FetchError: request to https://api.w3.org/specifications/2dcontext failed, reason: getaddrinfo ENOTFOUND api.w3.org api.w3.org:443 FetchError: request to https://api.w3.org/specifications/2dcontext failed, reason: getaddrinfo ENOTFOUND api.w3.org api.w3.org:443 at ClientRequest.&lt;anonymous&gt; (/home/dom/github.com/reffy/node_modules/node-fetch/index.js:133:11) at emitOne (events.js:96:13) at ClientRequest.emit (events.js:188:7) at TLSSocket.socketErrorListener (_http_client.js:309:9) at emitOne (events.js:96:13) at TLSSocket.emit (events.js:188:7) at connectErrorNT (net.js:1021:8) at _combinedTickCallback (internal/process/next_tick.js:80:11) at process._tickCallback (internal/process/next_tick.js:104:9)</code></li>
  <li><a href="https://www.w3.org/TR/accelerometer">accelerometer</a>: <code>FetchError: request to https://api.w3.org/specifications/accelerometer failed, reason: getaddrinfo ENOTFOUND api.w3.org api.w3.org:443 FetchError: request to https://api.w3.org/specifications/accelerometer failed, reason: getaddrinfo ENOTFOUND api.w3.org api.w3.org:443 at ClientRequest.&lt;anonymous&gt; (/home/dom/github.com/reffy/node_modules/node-fetch/index.js:133:11) at emitOne (events.js:96:13) at ClientRequest.emit (events.js:188:7) at TLSSocket.socketErrorListener (_http_client.js:309:9) at emitOne (events.js:96:13) at TLSSocket.emit (events.js:188:7) at connectErrorNT (net.js:1021:8) at _combinedTickCallback (internal/process/next_tick.js:80:11) at process._tickCallback (internal/process/next_tick.js:104:9)</code></li>
  <li><a href="https://www.w3.org/TR/ambient-light">ambient-light</a>: <code>FetchError: request to https://api.w3.org/specifications/ambient-light failed, reason: getaddrinfo ENOTFOUND api.w3.org api.w3.org:443 FetchError: request to https://api.w3.org/specifications/ambient-light failed, reason: getaddrinfo ENOTFOUND api.w3.org api.w3.org:443 at ClientRequest.&lt;anonymous&gt; (/home/dom/github.com/reffy/node_modules/node-fetch/index.js:133:11) at emitOne (events.js:96:13) at ClientRequest.emit (events.js:188:7) at TLSSocket.socketErrorListener (_http_client.js:309:9) at emitOne (events.js:96:13) at TLSSocket.emit (events.js:188:7) at connectErrorNT (net.js:1021:8) at _combinedTickCallback (internal/process/next_tick.js:80:11) at process._tickCallback (internal/process/next_tick.js:104:9)</code></li>
  <li><a href="https://www.w3.org/TR/appmanifest">appmanifest</a>: <code>FetchError: request to https://api.w3.org/specifications/appmanifest failed, reason: getaddrinfo ENOTFOUND api.w3.org api.w3.org:443 FetchError: request to https://api.w3.org/specifications/appmanifest failed, reason: getaddrinfo ENOTFOUND api.w3.org api.w3.org:443 at ClientRequest.&lt;anonymous&gt; (/home/dom/github.com/reffy/node_modules/node-fetch/index.js:133:11) at emitOne (events.js:96:13) at ClientRequest.emit (events.js:188:7) at TLSSocket.socketErrorListener (_http_client.js:309:9) at emitOne (events.js:96:13) at TLSSocket.emit (events.js:188:7) at connectErrorNT (net.js:1021:8) at _combinedTickCallback (internal/process/next_tick.js:80:11) at process._tickCallback (internal/process/next_tick.js:104:9)</code></li>
  <li><a href="https://www.w3.org/TR/audio-output">audio-output</a>: <code>FetchError: request to https://api.w3.org/specifications/audio-output failed, reason: getaddrinfo ENOTFOUND api.w3.org api.w3.org:443 FetchError: request to https://api.w3.org/specifications/audio-output failed, reason: getaddrinfo ENOTFOUND api.w3.org api.w3.org:443 at ClientRequest.&lt;anonymous&gt; (/home/dom/github.com/reffy/node_modules/node-fetch/index.js:133:11) at emitOne (events.js:96:13) at ClientRequest.emit (events.js:188:7) at TLSSocket.socketErrorListener (_http_client.js:309:9) at emitOne (events.js:96:13) at TLSSocket.emit (events.js:188:7) at connectErrorNT (net.js:1021:8) at _combinedTickCallback (internal/process/next_tick.js:80:11) at process._tickCallback (internal/process/next_tick.js:104:9)</code></li>
  <li><a href="https://www.w3.org/TR/battery-status">battery-status</a>: <code>FetchError: request to https://api.w3.org/specifications/battery-status failed, reason: getaddrinfo ENOTFOUND api.w3.org api.w3.org:443 FetchError: request to https://api.w3.org/specifications/battery-status failed, reason: getaddrinfo ENOTFOUND api.w3.org api.w3.org:443 at ClientRequest.&lt;anonymous&gt; (/home/dom/github.com/reffy/node_modules/node-fetch/index.js:133:11) at emitOne (events.js:96:13) at ClientRequest.emit (events.js:188:7) at TLSSocket.socketErrorListener (_http_client.js:309:9) at emitOne (events.js:96:13) at TLSSocket.emit (events.js:188:7) at connectErrorNT (net.js:1021:8) at _combinedTickCallback (internal/process/next_tick.js:80:11) at process._tickCallback (internal/process/next_tick.js:104:9)</code></li>
  <li><a href="https://www.w3.org/TR/beacon">beacon</a>: <code>FetchError: request to https://api.w3.org/specifications/beacon failed, reason: getaddrinfo ENOTFOUND api.w3.org api.w3.org:443 FetchError: request to https://api.w3.org/specifications/beacon failed, reason: getaddrinfo ENOTFOUND api.w3.org api.w3.org:443 at ClientRequest.&lt;anonymous&gt; (/home/dom/github.com/reffy/node_modules/node-fetch/index.js:133:11) at emitOne (events.js:96:13) at ClientRequest.emit (events.js:188:7) at TLSSocket.socketErrorListener (_http_client.js:309:9) at emitOne (events.js:96:13) at TLSSocket.emit (events.js:188:7) at connectErrorNT (net.js:1021:8) at _combinedTickCallback (internal/process/next_tick.js:80:11) at process._tickCallback (internal/process/next_tick.js:104:9)</code></li>
  <li><a href="https://www.w3.org/TR/clipboard-apis">clipboard-apis</a>: <code>FetchError: request to https://api.w3.org/specifications/clipboard-apis failed, reason: getaddrinfo ENOTFOUND api.w3.org api.w3.org:443 FetchError: request to https://api.w3.org/specifications/clipboard-apis failed, reason: getaddrinfo ENOTFOUND api.w3.org api.w3.org:443 at ClientRequest.&lt;anonymous&gt; (/home/dom/github.com/reffy/node_modules/node-fetch/index.js:133:11) at emitOne (events.js:96:13) at ClientRequest.emit (events.js:188:7) at TLSSocket.socketErrorListener (_http_client.js:309:9) at emitOne (events.js:96:13) at TLSSocket.emit (events.js:188:7) at connectErrorNT (net.js:1021:8) at _combinedTickCallback (internal/process/next_tick.js:80:11) at process._tickCallback (internal/process/next_tick.js:104:9)</code></li>
  <li><a href="https://www.w3.org/TR/compositing-1">compositing-1</a>: <code>FetchError: request to https://api.w3.org/specifications/compositing-1 failed, reason: getaddrinfo ENOTFOUND api.w3.org api.w3.org:443 FetchError: request to https://api.w3.org/specifications/compositing-1 failed, reason: getaddrinfo ENOTFOUND api.w3.org api.w3.org:443 at ClientRequest.&lt;anonymous&gt; (/home/dom/github.com/reffy/node_modules/node-fetch/index.js:133:11) at emitOne (events.js:96:13) at ClientRequest.emit (events.js:188:7) at TLSSocket.socketErrorListener (_http_client.js:309:9) at emitOne (events.js:96:13) at TLSSocket.emit (events.js:188:7) at connectErrorNT (net.js:1021:8) at _combinedTickCallback (internal/process/next_tick.js:80:11) at process._tickCallback (internal/process/next_tick.js:104:9)</code></li>
  <li><a href="https://www.w3.org/TR/credential-management-1">credential-management-1</a>: <code>FetchError: request to https://api.w3.org/specifications/credential-management-1 failed, reason: getaddrinfo ENOTFOUND api.w3.org api.w3.org:443 FetchError: request to https://api.w3.org/specifications/credential-management-1 failed, reason: getaddrinfo ENOTFOUND api.w3.org api.w3.org:443 at ClientRequest.&lt;anonymous&gt; (/home/dom/github.com/reffy/node_modules/node-fetch/index.js:133:11) at emitOne (events.js:96:13) at ClientRequest.emit (events.js:188:7) at TLSSocket.socketErrorListener (_http_client.js:309:9) at emitOne (events.js:96:13) at TLSSocket.emit (events.js:188:7) at connectErrorNT (net.js:1021:8) at _combinedTickCallback (internal/process/next_tick.js:80:11) at process._tickCallback (internal/process/next_tick.js:104:9)</code></li>
  <li><a href="https://www.w3.org/TR/CSP3">CSP3</a>: <code>FetchError: request to https://api.w3.org/specifications/CSP3 failed, reason: getaddrinfo ENOTFOUND api.w3.org api.w3.org:443 FetchError: request to https://api.w3.org/specifications/CSP3 failed, reason: getaddrinfo ENOTFOUND api.w3.org api.w3.org:443 at ClientRequest.&lt;anonymous&gt; (/home/dom/github.com/reffy/node_modules/node-fetch/index.js:133:11) at emitOne (events.js:96:13) at ClientRequest.emit (events.js:188:7) at TLSSocket.socketErrorListener (_http_client.js:309:9) at emitOne (events.js:96:13) at TLSSocket.emit (events.js:188:7) at connectErrorNT (net.js:1021:8) at _combinedTickCallback (internal/process/next_tick.js:80:11) at process._tickCallback (internal/process/next_tick.js:104:9)</code></li>
  <li><a href="https://www.w3.org/TR/css-font-loading-3">css-font-loading-3</a>: <code>FetchError: request to https://api.w3.org/specifications/css-font-loading-3 failed, reason: getaddrinfo ENOTFOUND api.w3.org api.w3.org:443 FetchError: request to https://api.w3.org/specifications/css-font-loading-3 failed, reason: getaddrinfo ENOTFOUND api.w3.org api.w3.org:443 at ClientRequest.&lt;anonymous&gt; (/home/dom/github.com/reffy/node_modules/node-fetch/index.js:133:11) at emitOne (events.js:96:13) at ClientRequest.emit (events.js:188:7) at TLSSocket.socketErrorListener (_http_client.js:309:9) at emitOne (events.js:96:13) at TLSSocket.emit (events.js:188:7) at connectErrorNT (net.js:1021:8) at _combinedTickCallback (internal/process/next_tick.js:80:11) at process._tickCallback (internal/process/next_tick.js:104:9)</code></li>
  <li><a href="https://www.w3.org/TR/css-fonts-3">css-fonts-3</a>: <code>FetchError: request to https://api.w3.org/specifications/css-fonts-3 failed, reason: getaddrinfo ENOTFOUND api.w3.org api.w3.org:443 FetchError: request to https://api.w3.org/specifications/css-fonts-3 failed, reason: getaddrinfo ENOTFOUND api.w3.org api.w3.org:443 at ClientRequest.&lt;anonymous&gt; (/home/dom/github.com/reffy/node_modules/node-fetch/index.js:133:11) at emitOne (events.js:96:13) at ClientRequest.emit (events.js:188:7) at TLSSocket.socketErrorListener (_http_client.js:309:9) at emitOne (events.js:96:13) at TLSSocket.emit (events.js:188:7) at connectErrorNT (net.js:1021:8) at _combinedTickCallback (internal/process/next_tick.js:80:11) at process._tickCallback (internal/process/next_tick.js:104:9)</code></li>
  <li><a href="https://www.w3.org/TR/css-paint-api-1">css-paint-api-1</a>: <code>FetchError: request to https://api.w3.org/specifications/css-paint-api-1 failed, reason: getaddrinfo ENOTFOUND api.w3.org api.w3.org:443 FetchError: request to https://api.w3.org/specifications/css-paint-api-1 failed, reason: getaddrinfo ENOTFOUND api.w3.org api.w3.org:443 at ClientRequest.&lt;anonymous&gt; (/home/dom/github.com/reffy/node_modules/node-fetch/index.js:133:11) at emitOne (events.js:96:13) at ClientRequest.emit (events.js:188:7) at TLSSocket.socketErrorListener (_http_client.js:309:9) at emitOne (events.js:96:13) at TLSSocket.emit (events.js:188:7) at connectErrorNT (net.js:1021:8) at _combinedTickCallback (internal/process/next_tick.js:80:11) at process._tickCallback (internal/process/next_tick.js:104:9)</code></li>
  <li><a href="https://www.w3.org/TR/css-properties-values-api-1">css-properties-values-api-1</a>: <code>FetchError: request to https://api.w3.org/specifications/css-properties-values-api-1 failed, reason: getaddrinfo ENOTFOUND api.w3.org api.w3.org:443 FetchError: request to https://api.w3.org/specifications/css-properties-values-api-1 failed, reason: getaddrinfo ENOTFOUND api.w3.org api.w3.org:443 at ClientRequest.&lt;anonymous&gt; (/home/dom/github.com/reffy/node_modules/node-fetch/index.js:133:11) at emitOne (events.js:96:13) at ClientRequest.emit (events.js:188:7) at TLSSocket.socketErrorListener (_http_client.js:309:9) at emitOne (events.js:96:13) at TLSSocket.emit (events.js:188:7) at connectErrorNT (net.js:1021:8) at _combinedTickCallback (internal/process/next_tick.js:80:11) at process._tickCallback (internal/process/next_tick.js:104:9)</code></li>
  <li><a href="https://www.w3.org/TR/css-pseudo-4">css-pseudo-4</a>: <code>FetchError: request to https://api.w3.org/specifications/css-pseudo-4 failed, reason: getaddrinfo ENOTFOUND api.w3.org api.w3.org:443 FetchError: request to https://api.w3.org/specifications/css-pseudo-4 failed, reason: getaddrinfo ENOTFOUND api.w3.org api.w3.org:443 at ClientRequest.&lt;anonymous&gt; (/home/dom/github.com/reffy/node_modules/node-fetch/index.js:133:11) at emitOne (events.js:96:13) at ClientRequest.emit (events.js:188:7) at TLSSocket.socketErrorListener (_http_client.js:309:9) at emitOne (events.js:96:13) at TLSSocket.emit (events.js:188:7) at connectErrorNT (net.js:1021:8) at _combinedTickCallback (internal/process/next_tick.js:80:11) at process._tickCallback (internal/process/next_tick.js:104:9)</code></li>
  <li><a href="https://www.w3.org/TR/css-syntax-3">css-syntax-3</a>: <code>FetchError: request to https://api.w3.org/specifications/css-syntax-3 failed, reason: getaddrinfo ENOTFOUND api.w3.org api.w3.org:443 FetchError: request to https://api.w3.org/specifications/css-syntax-3 failed, reason: getaddrinfo ENOTFOUND api.w3.org api.w3.org:443 at ClientRequest.&lt;anonymous&gt; (/home/dom/github.com/reffy/node_modules/node-fetch/index.js:133:11) at emitOne (events.js:96:13) at ClientRequest.emit (events.js:188:7) at TLSSocket.socketErrorListener (_http_client.js:309:9) at emitOne (events.js:96:13) at TLSSocket.emit (events.js:188:7) at connectErrorNT (net.js:1021:8) at _combinedTickCallback (internal/process/next_tick.js:80:11) at process._tickCallback (internal/process/next_tick.js:104:9)</code></li>
  <li><a href="https://www.w3.org/TR/css-typed-om-1">css-typed-om-1</a>: <code>FetchError: request to https://api.w3.org/specifications/css-typed-om-1 failed, reason: getaddrinfo ENOTFOUND api.w3.org api.w3.org:443 FetchError: request to https://api.w3.org/specifications/css-typed-om-1 failed, reason: getaddrinfo ENOTFOUND api.w3.org api.w3.org:443 at ClientRequest.&lt;anonymous&gt; (/home/dom/github.com/reffy/node_modules/node-fetch/index.js:133:11) at emitOne (events.js:96:13) at ClientRequest.emit (events.js:188:7) at TLSSocket.socketErrorListener (_http_client.js:309:9) at emitOne (events.js:96:13) at TLSSocket.emit (events.js:188:7) at connectErrorNT (net.js:1021:8) at _combinedTickCallback (internal/process/next_tick.js:80:11) at process._tickCallback (internal/process/next_tick.js:104:9)</code></li>
  <li><a href="https://www.w3.org/TR/css-values-3">css-values-3</a>: <code>FetchError: request to https://api.w3.org/specifications/css-values-3 failed, reason: getaddrinfo ENOTFOUND api.w3.org api.w3.org:443 FetchError: request to https://api.w3.org/specifications/css-values-3 failed, reason: getaddrinfo ENOTFOUND api.w3.org api.w3.org:443 at ClientRequest.&lt;anonymous&gt; (/home/dom/github.com/reffy/node_modules/node-fetch/index.js:133:11) at emitOne (events.js:96:13) at ClientRequest.emit (events.js:188:7) at TLSSocket.socketErrorListener (_http_client.js:309:9) at emitOne (events.js:96:13) at TLSSocket.emit (events.js:188:7) at connectErrorNT (net.js:1021:8) at _combinedTickCallback (internal/process/next_tick.js:80:11) at process._tickCallback (internal/process/next_tick.js:104:9)</code></li>
  <li><a href="https://www.w3.org/TR/css-writing-modes-3">css-writing-modes-3</a>: <code>FetchError: request to https://api.w3.org/specifications/css-writing-modes-3 failed, reason: getaddrinfo ENOTFOUND api.w3.org api.w3.org:443 FetchError: request to https://api.w3.org/specifications/css-writing-modes-3 failed, reason: getaddrinfo ENOTFOUND api.w3.org api.w3.org:443 at ClientRequest.&lt;anonymous&gt; (/home/dom/github.com/reffy/node_modules/node-fetch/index.js:133:11) at emitOne (events.js:96:13) at ClientRequest.emit (events.js:188:7) at TLSSocket.socketErrorListener (_http_client.js:309:9) at emitOne (events.js:96:13) at TLSSocket.emit (events.js:188:7) at connectErrorNT (net.js:1021:8) at _combinedTickCallback (internal/process/next_tick.js:80:11) at process._tickCallback (internal/process/next_tick.js:104:9)</code></li>
  <li><a href="https://www.w3.org/TR/css3-selectors">css3-selectors</a>: <code>FetchError: request to https://api.w3.org/specifications/css3-selectors failed, reason: getaddrinfo ENOTFOUND api.w3.org api.w3.org:443 FetchError: request to https://api.w3.org/specifications/css3-selectors failed, reason: getaddrinfo ENOTFOUND api.w3.org api.w3.org:443 at ClientRequest.&lt;anonymous&gt; (/home/dom/github.com/reffy/node_modules/node-fetch/index.js:133:11) at emitOne (events.js:96:13) at ClientRequest.emit (events.js:188:7) at TLSSocket.socketErrorListener (_http_client.js:309:9) at emitOne (events.js:96:13) at TLSSocket.emit (events.js:188:7) at connectErrorNT (net.js:1021:8) at _combinedTickCallback (internal/process/next_tick.js:80:11) at process._tickCallback (internal/process/next_tick.js:104:9)</code></li>
  <li><a href="https://www.w3.org/TR/cssom-1">cssom-1</a>: <code>FetchError: request to https://api.w3.org/specifications/cssom-1 failed, reason: getaddrinfo ENOTFOUND api.w3.org api.w3.org:443 FetchError: request to https://api.w3.org/specifications/cssom-1 failed, reason: getaddrinfo ENOTFOUND api.w3.org api.w3.org:443 at ClientRequest.&lt;anonymous&gt; (/home/dom/github.com/reffy/node_modules/node-fetch/index.js:133:11) at emitOne (events.js:96:13) at ClientRequest.emit (events.js:188:7) at TLSSocket.socketErrorListener (_http_client.js:309:9) at emitOne (events.js:96:13) at TLSSocket.emit (events.js:188:7) at connectErrorNT (net.js:1021:8) at _combinedTickCallback (internal/process/next_tick.js:80:11) at process._tickCallback (internal/process/next_tick.js:104:9)</code></li>
  <li><a href="https://www.w3.org/TR/cssom-view-1">cssom-view-1</a>: <code>FetchError: request to https://api.w3.org/specifications/cssom-view-1 failed, reason: getaddrinfo ENOTFOUND api.w3.org api.w3.org:443 FetchError: request to https://api.w3.org/specifications/cssom-view-1 failed, reason: getaddrinfo ENOTFOUND api.w3.org api.w3.org:443 at ClientRequest.&lt;anonymous&gt; (/home/dom/github.com/reffy/node_modules/node-fetch/index.js:133:11) at emitOne (events.js:96:13) at ClientRequest.emit (events.js:188:7) at TLSSocket.socketErrorListener (_http_client.js:309:9) at emitOne (events.js:96:13) at TLSSocket.emit (events.js:188:7) at connectErrorNT (net.js:1021:8) at _combinedTickCallback (internal/process/next_tick.js:80:11) at process._tickCallback (internal/process/next_tick.js:104:9)</code></li>
  <li><a href="https://www.w3.org/TR/custom-elements">custom-elements</a>: <code>FetchError: request to https://api.w3.org/specifications/custom-elements failed, reason: getaddrinfo ENOTFOUND api.w3.org api.w3.org:443 FetchError: request to https://api.w3.org/specifications/custom-elements failed, reason: getaddrinfo ENOTFOUND api.w3.org api.w3.org:443 at ClientRequest.&lt;anonymous&gt; (/home/dom/github.com/reffy/node_modules/node-fetch/index.js:133:11) at emitOne (events.js:96:13) at ClientRequest.emit (events.js:188:7) at TLSSocket.socketErrorListener (_http_client.js:309:9) at emitOne (events.js:96:13) at TLSSocket.emit (events.js:188:7) at connectErrorNT (net.js:1021:8) at _combinedTickCallback (internal/process/next_tick.js:80:11) at process._tickCallback (internal/process/next_tick.js:104:9)</code></li>
  <li><a href="https://www.w3.org/TR/DOM-Parsing">DOM-Parsing</a>: <code>FetchError: request to https://api.w3.org/specifications/DOM-Parsing failed, reason: getaddrinfo ENOTFOUND api.w3.org api.w3.org:443 FetchError: request to https://api.w3.org/specifications/DOM-Parsing failed, reason: getaddrinfo ENOTFOUND api.w3.org api.w3.org:443 at ClientRequest.&lt;anonymous&gt; (/home/dom/github.com/reffy/node_modules/node-fetch/index.js:133:11) at emitOne (events.js:96:13) at ClientRequest.emit (events.js:188:7) at TLSSocket.socketErrorListener (_http_client.js:309:9) at emitOne (events.js:96:13) at TLSSocket.emit (events.js:188:7) at connectErrorNT (net.js:1021:8) at _combinedTickCallback (internal/process/next_tick.js:80:11) at process._tickCallback (internal/process/next_tick.js:104:9)</code></li>
  <li><a href="https://www.w3.org/TR/dom41">dom41</a>: <code>FetchError: request to https://api.w3.org/specifications/dom41 failed, reason: getaddrinfo ENOTFOUND api.w3.org api.w3.org:443 FetchError: request to https://api.w3.org/specifications/dom41 failed, reason: getaddrinfo ENOTFOUND api.w3.org api.w3.org:443 at ClientRequest.&lt;anonymous&gt; (/home/dom/github.com/reffy/node_modules/node-fetch/index.js:133:11) at emitOne (events.js:96:13) at ClientRequest.emit (events.js:188:7) at TLSSocket.socketErrorListener (_http_client.js:309:9) at emitOne (events.js:96:13) at TLSSocket.emit (events.js:188:7) at connectErrorNT (net.js:1021:8) at _combinedTickCallback (internal/process/next_tick.js:80:11) at process._tickCallback (internal/process/next_tick.js:104:9)</code></li>
  <li><a href="https://www.w3.org/TR/ElementTraversal">ElementTraversal</a>: <code>FetchError: request to https://api.w3.org/specifications/ElementTraversal failed, reason: getaddrinfo ENOTFOUND api.w3.org api.w3.org:443 FetchError: request to https://api.w3.org/specifications/ElementTraversal failed, reason: getaddrinfo ENOTFOUND api.w3.org api.w3.org:443 at ClientRequest.&lt;anonymous&gt; (/home/dom/github.com/reffy/node_modules/node-fetch/index.js:133:11) at emitOne (events.js:96:13) at ClientRequest.emit (events.js:188:7) at TLSSocket.socketErrorListener (_http_client.js:309:9) at emitOne (events.js:96:13) at TLSSocket.emit (events.js:188:7) at connectErrorNT (net.js:1021:8) at _combinedTickCallback (internal/process/next_tick.js:80:11) at process._tickCallback (internal/process/next_tick.js:104:9)</code></li>
  <li><a href="https://www.w3.org/TR/encoding">encoding</a>: <code>FetchError: request to https://api.w3.org/specifications/encoding failed, reason: getaddrinfo ENOTFOUND api.w3.org api.w3.org:443 FetchError: request to https://api.w3.org/specifications/encoding failed, reason: getaddrinfo ENOTFOUND api.w3.org api.w3.org:443 at ClientRequest.&lt;anonymous&gt; (/home/dom/github.com/reffy/node_modules/node-fetch/index.js:133:11) at emitOne (events.js:96:13) at ClientRequest.emit (events.js:188:7) at TLSSocket.socketErrorListener (_http_client.js:309:9) at emitOne (events.js:96:13) at TLSSocket.emit (events.js:188:7) at connectErrorNT (net.js:1021:8) at _combinedTickCallback (internal/process/next_tick.js:80:11) at process._tickCallback (internal/process/next_tick.js:104:9)</code></li>
  <li><a href="https://www.w3.org/TR/encrypted-media">encrypted-media</a>: <code>FetchError: request to https://api.w3.org/specifications/encrypted-media failed, reason: getaddrinfo ENOTFOUND api.w3.org api.w3.org:443 FetchError: request to https://api.w3.org/specifications/encrypted-media failed, reason: getaddrinfo ENOTFOUND api.w3.org api.w3.org:443 at ClientRequest.&lt;anonymous&gt; (/home/dom/github.com/reffy/node_modules/node-fetch/index.js:133:11) at emitOne (events.js:96:13) at ClientRequest.emit (events.js:188:7) at TLSSocket.socketErrorListener (_http_client.js:309:9) at emitOne (events.js:96:13) at TLSSocket.emit (events.js:188:7) at connectErrorNT (net.js:1021:8) at _combinedTickCallback (internal/process/next_tick.js:80:11) at process._tickCallback (internal/process/next_tick.js:104:9)</code></li>
  <li><a href="https://www.w3.org/TR/eventsource">eventsource</a>: <code>FetchError: request to https://api.w3.org/specifications/eventsource failed, reason: getaddrinfo ENOTFOUND api.w3.org api.w3.org:443 FetchError: request to https://api.w3.org/specifications/eventsource failed, reason: getaddrinfo ENOTFOUND api.w3.org api.w3.org:443 at ClientRequest.&lt;anonymous&gt; (/home/dom/github.com/reffy/node_modules/node-fetch/index.js:133:11) at emitOne (events.js:96:13) at ClientRequest.emit (events.js:188:7) at TLSSocket.socketErrorListener (_http_client.js:309:9) at emitOne (events.js:96:13) at TLSSocket.emit (events.js:188:7) at connectErrorNT (net.js:1021:8) at _combinedTickCallback (internal/process/next_tick.js:80:11) at process._tickCallback (internal/process/next_tick.js:104:9)</code></li>
  <li><a href="https://www.w3.org/TR/FileAPI">FileAPI</a>: <code>FetchError: request to https://api.w3.org/specifications/FileAPI failed, reason: getaddrinfo ENOTFOUND api.w3.org api.w3.org:443 FetchError: request to https://api.w3.org/specifications/FileAPI failed, reason: getaddrinfo ENOTFOUND api.w3.org api.w3.org:443 at ClientRequest.&lt;anonymous&gt; (/home/dom/github.com/reffy/node_modules/node-fetch/index.js:133:11) at emitOne (events.js:96:13) at ClientRequest.emit (events.js:188:7) at TLSSocket.socketErrorListener (_http_client.js:309:9) at emitOne (events.js:96:13) at TLSSocket.emit (events.js:188:7) at connectErrorNT (net.js:1021:8) at _combinedTickCallback (internal/process/next_tick.js:80:11) at process._tickCallback (internal/process/next_tick.js:104:9)</code></li>
  <li><a href="https://www.w3.org/TR/gamepad">gamepad</a>: <code>FetchError: request to https://api.w3.org/specifications/gamepad failed, reason: getaddrinfo ENOTFOUND api.w3.org api.w3.org:443 FetchError: request to https://api.w3.org/specifications/gamepad failed, reason: getaddrinfo ENOTFOUND api.w3.org api.w3.org:443 at ClientRequest.&lt;anonymous&gt; (/home/dom/github.com/reffy/node_modules/node-fetch/index.js:133:11) at emitOne (events.js:96:13) at ClientRequest.emit (events.js:188:7) at TLSSocket.socketErrorListener (_http_client.js:309:9) at emitOne (events.js:96:13) at TLSSocket.emit (events.js:188:7) at connectErrorNT (net.js:1021:8) at _combinedTickCallback (internal/process/next_tick.js:80:11) at process._tickCallback (internal/process/next_tick.js:104:9)</code></li>
  <li><a href="https://www.w3.org/TR/generic-sensor">generic-sensor</a>: <code>FetchError: request to https://api.w3.org/specifications/generic-sensor failed, reason: getaddrinfo ENOTFOUND api.w3.org api.w3.org:443 FetchError: request to https://api.w3.org/specifications/generic-sensor failed, reason: getaddrinfo ENOTFOUND api.w3.org api.w3.org:443 at ClientRequest.&lt;anonymous&gt; (/home/dom/github.com/reffy/node_modules/node-fetch/index.js:133:11) at emitOne (events.js:96:13) at ClientRequest.emit (events.js:188:7) at TLSSocket.socketErrorListener (_http_client.js:309:9) at emitOne (events.js:96:13) at TLSSocket.emit (events.js:188:7) at connectErrorNT (net.js:1021:8) at _combinedTickCallback (internal/process/next_tick.js:80:11) at process._tickCallback (internal/process/next_tick.js:104:9)</code></li>
  <li><a href="https://www.w3.org/TR/geolocation-API">geolocation-API</a>: <code>FetchError: request to https://api.w3.org/specifications/geolocation-API failed, reason: getaddrinfo ENOTFOUND api.w3.org api.w3.org:443 FetchError: request to https://api.w3.org/specifications/geolocation-API failed, reason: getaddrinfo ENOTFOUND api.w3.org api.w3.org:443 at ClientRequest.&lt;anonymous&gt; (/home/dom/github.com/reffy/node_modules/node-fetch/index.js:133:11) at emitOne (events.js:96:13) at ClientRequest.emit (events.js:188:7) at TLSSocket.socketErrorListener (_http_client.js:309:9) at emitOne (events.js:96:13) at TLSSocket.emit (events.js:188:7) at connectErrorNT (net.js:1021:8) at _combinedTickCallback (internal/process/next_tick.js:80:11) at process._tickCallback (internal/process/next_tick.js:104:9)</code></li>
  <li><a href="https://www.w3.org/TR/geometry-1">geometry-1</a>: <code>FetchError: request to https://api.w3.org/specifications/geometry-1 failed, reason: getaddrinfo ENOTFOUND api.w3.org api.w3.org:443 FetchError: request to https://api.w3.org/specifications/geometry-1 failed, reason: getaddrinfo ENOTFOUND api.w3.org api.w3.org:443 at ClientRequest.&lt;anonymous&gt; (/home/dom/github.com/reffy/node_modules/node-fetch/index.js:133:11) at emitOne (events.js:96:13) at ClientRequest.emit (events.js:188:7) at TLSSocket.socketErrorListener (_http_client.js:309:9) at emitOne (events.js:96:13) at TLSSocket.emit (events.js:188:7) at connectErrorNT (net.js:1021:8) at _combinedTickCallback (internal/process/next_tick.js:80:11) at process._tickCallback (internal/process/next_tick.js:104:9)</code></li>
  <li><a href="https://www.w3.org/TR/gyroscope">gyroscope</a>: <code>FetchError: request to https://api.w3.org/specifications/gyroscope failed, reason: getaddrinfo ENOTFOUND api.w3.org api.w3.org:443 FetchError: request to https://api.w3.org/specifications/gyroscope failed, reason: getaddrinfo ENOTFOUND api.w3.org api.w3.org:443 at ClientRequest.&lt;anonymous&gt; (/home/dom/github.com/reffy/node_modules/node-fetch/index.js:133:11) at emitOne (events.js:96:13) at ClientRequest.emit (events.js:188:7) at TLSSocket.socketErrorListener (_http_client.js:309:9) at emitOne (events.js:96:13) at TLSSocket.emit (events.js:188:7) at connectErrorNT (net.js:1021:8) at _combinedTickCallback (internal/process/next_tick.js:80:11) at process._tickCallback (internal/process/next_tick.js:104:9)</code></li>
  <li><a href="https://www.w3.org/TR/hr-time-3">hr-time-3</a>: <code>FetchError: request to https://api.w3.org/specifications/hr-time-3 failed, reason: getaddrinfo ENOTFOUND api.w3.org api.w3.org:443 FetchError: request to https://api.w3.org/specifications/hr-time-3 failed, reason: getaddrinfo ENOTFOUND api.w3.org api.w3.org:443 at ClientRequest.&lt;anonymous&gt; (/home/dom/github.com/reffy/node_modules/node-fetch/index.js:133:11) at emitOne (events.js:96:13) at ClientRequest.emit (events.js:188:7) at TLSSocket.socketErrorListener (_http_client.js:309:9) at emitOne (events.js:96:13) at TLSSocket.emit (events.js:188:7) at connectErrorNT (net.js:1021:8) at _combinedTickCallback (internal/process/next_tick.js:80:11) at process._tickCallback (internal/process/next_tick.js:104:9)</code></li>
  <li><a href="https://www.w3.org/TR/html-media-capture">html-media-capture</a>: <code>FetchError: request to https://api.w3.org/specifications/html-media-capture failed, reason: getaddrinfo ENOTFOUND api.w3.org api.w3.org:443 FetchError: request to https://api.w3.org/specifications/html-media-capture failed, reason: getaddrinfo ENOTFOUND api.w3.org api.w3.org:443 at ClientRequest.&lt;anonymous&gt; (/home/dom/github.com/reffy/node_modules/node-fetch/index.js:133:11) at emitOne (events.js:96:13) at ClientRequest.emit (events.js:188:7) at TLSSocket.socketErrorListener (_http_client.js:309:9) at emitOne (events.js:96:13) at TLSSocket.emit (events.js:188:7) at connectErrorNT (net.js:1021:8) at _combinedTickCallback (internal/process/next_tick.js:80:11) at process._tickCallback (internal/process/next_tick.js:104:9)</code></li>
  <li><a href="https://www.w3.org/TR/html52">html52</a>: <code>FetchError: request to https://api.w3.org/specifications/html52 failed, reason: getaddrinfo ENOTFOUND api.w3.org api.w3.org:443 FetchError: request to https://api.w3.org/specifications/html52 failed, reason: getaddrinfo ENOTFOUND api.w3.org api.w3.org:443 at ClientRequest.&lt;anonymous&gt; (/home/dom/github.com/reffy/node_modules/node-fetch/index.js:133:11) at emitOne (events.js:96:13) at ClientRequest.emit (events.js:188:7) at TLSSocket.socketErrorListener (_http_client.js:309:9) at emitOne (events.js:96:13) at TLSSocket.emit (events.js:188:7) at connectErrorNT (net.js:1021:8) at _combinedTickCallback (internal/process/next_tick.js:80:11) at process._tickCallback (internal/process/next_tick.js:104:9)</code></li>
  <li><a href="https://console.spec.whatwg.org/">https://console.spec.whatwg.org/</a>: <code>FetchError: request to https://console.spec.whatwg.org/ failed, reason: getaddrinfo ENOTFOUND console.spec.whatwg.org console.spec.whatwg.org:443 FetchError: request to https://console.spec.whatwg.org/ failed, reason: getaddrinfo ENOTFOUND console.spec.whatwg.org console.spec.whatwg.org:443 at ClientRequest.&lt;anonymous&gt; (/home/dom/github.com/reffy/node_modules/node-fetch/index.js:133:11) at emitOne (events.js:96:13) at ClientRequest.emit (events.js:188:7) at TLSSocket.socketErrorListener (_http_client.js:309:9) at emitOne (events.js:96:13) at TLSSocket.emit (events.js:188:7) at connectErrorNT (net.js:1021:8) at _combinedTickCallback (internal/process/next_tick.js:80:11) at process._tickCallback (internal/process/next_tick.js:104:9)</code></li>
  <li><a href="https://fetch.spec.whatwg.org/">https://fetch.spec.whatwg.org/</a>: <code>FetchError: request to https://fetch.spec.whatwg.org/ failed, reason: getaddrinfo ENOTFOUND fetch.spec.whatwg.org fetch.spec.whatwg.org:443 FetchError: request to https://fetch.spec.whatwg.org/ failed, reason: getaddrinfo ENOTFOUND fetch.spec.whatwg.org fetch.spec.whatwg.org:443 at ClientRequest.&lt;anonymous&gt; (/home/dom/github.com/reffy/node_modules/node-fetch/index.js:133:11) at emitOne (events.js:96:13) at ClientRequest.emit (events.js:188:7) at TLSSocket.socketErrorListener (_http_client.js:309:9) at emitOne (events.js:96:13) at TLSSocket.emit (events.js:188:7) at connectErrorNT (net.js:1021:8) at _combinedTickCallback (internal/process/next_tick.js:80:11) at process._tickCallback (internal/process/next_tick.js:104:9)</code></li>
  <li><a href="https://fullscreen.spec.whatwg.org/">https://fullscreen.spec.whatwg.org/</a>: <code>FetchError: request to https://fullscreen.spec.whatwg.org/ failed, reason: getaddrinfo ENOTFOUND fullscreen.spec.whatwg.org fullscreen.spec.whatwg.org:443 FetchError: request to https://fullscreen.spec.whatwg.org/ failed, reason: getaddrinfo ENOTFOUND fullscreen.spec.whatwg.org fullscreen.spec.whatwg.org:443 at ClientRequest.&lt;anonymous&gt; (/home/dom/github.com/reffy/node_modules/node-fetch/index.js:133:11) at emitOne (events.js:96:13) at ClientRequest.emit (events.js:188:7) at TLSSocket.socketErrorListener (_http_client.js:309:9) at emitOne (events.js:96:13) at TLSSocket.emit (events.js:188:7) at connectErrorNT (net.js:1021:8) at _combinedTickCallback (internal/process/next_tick.js:80:11) at process._tickCallback (internal/process/next_tick.js:104:9)</code></li>
  <li><a href="https://storage.spec.whatwg.org/">https://storage.spec.whatwg.org/</a>: <code>FetchError: request to https://storage.spec.whatwg.org/ failed, reason: getaddrinfo ENOTFOUND storage.spec.whatwg.org storage.spec.whatwg.org:443 FetchError: request to https://storage.spec.whatwg.org/ failed, reason: getaddrinfo ENOTFOUND storage.spec.whatwg.org storage.spec.whatwg.org:443 at ClientRequest.&lt;anonymous&gt; (/home/dom/github.com/reffy/node_modules/node-fetch/index.js:133:11) at emitOne (events.js:96:13) at ClientRequest.emit (events.js:188:7) at TLSSocket.socketErrorListener (_http_client.js:309:9) at emitOne (events.js:96:13) at TLSSocket.emit (events.js:188:7) at connectErrorNT (net.js:1021:8) at _combinedTickCallback (internal/process/next_tick.js:80:11) at process._tickCallback (internal/process/next_tick.js:104:9)</code></li>
  <li><a href="https://streams.spec.whatwg.org/">https://streams.spec.whatwg.org/</a>: <code>FetchError: request to https://streams.spec.whatwg.org/ failed, reason: getaddrinfo ENOTFOUND streams.spec.whatwg.org streams.spec.whatwg.org:443 FetchError: request to https://streams.spec.whatwg.org/ failed, reason: getaddrinfo ENOTFOUND streams.spec.whatwg.org streams.spec.whatwg.org:443 at ClientRequest.&lt;anonymous&gt; (/home/dom/github.com/reffy/node_modules/node-fetch/index.js:133:11) at emitOne (events.js:96:13) at ClientRequest.emit (events.js:188:7) at TLSSocket.socketErrorListener (_http_client.js:309:9) at emitOne (events.js:96:13) at TLSSocket.emit (events.js:188:7) at connectErrorNT (net.js:1021:8) at _combinedTickCallback (internal/process/next_tick.js:80:11) at process._tickCallback (internal/process/next_tick.js:104:9)</code></li>
  <li><a href="https://url.spec.whatwg.org/">https://url.spec.whatwg.org/</a>: <code>FetchError: request to https://url.spec.whatwg.org/ failed, reason: getaddrinfo ENOTFOUND url.spec.whatwg.org url.spec.whatwg.org:443 FetchError: request to https://url.spec.whatwg.org/ failed, reason: getaddrinfo ENOTFOUND url.spec.whatwg.org url.spec.whatwg.org:443 at ClientRequest.&lt;anonymous&gt; (/home/dom/github.com/reffy/node_modules/node-fetch/index.js:133:11) at emitOne (events.js:96:13) at ClientRequest.emit (events.js:188:7) at TLSSocket.socketErrorListener (_http_client.js:309:9) at emitOne (events.js:96:13) at TLSSocket.emit (events.js:188:7) at connectErrorNT (net.js:1021:8) at _combinedTickCallback (internal/process/next_tick.js:80:11) at process._tickCallback (internal/process/next_tick.js:104:9)</code></li>
  <li><a href="https://www.khronos.org/registry/webgl/specs/latest/1.0/">https://www.khronos.org/registry/webgl/specs/latest/1.0/</a>: <code>FetchError: request to https://www.khronos.org/registry/webgl/specs/latest/1.0/ failed, reason: getaddrinfo ENOTFOUND www.khronos.org www.khronos.org:443 FetchError: request to https://www.khronos.org/registry/webgl/specs/latest/1.0/ failed, reason: getaddrinfo ENOTFOUND www.khronos.org www.khronos.org:443 at ClientRequest.&lt;anonymous&gt; (/home/dom/github.com/reffy/node_modules/node-fetch/index.js:133:11) at emitOne (events.js:96:13) at ClientRequest.emit (events.js:188:7) at TLSSocket.socketErrorListener (_http_client.js:309:9) at emitOne (events.js:96:13) at TLSSocket.emit (events.js:188:7) at connectErrorNT (net.js:1021:8) at _combinedTickCallback (internal/process/next_tick.js:80:11) at process._tickCallback (internal/process/next_tick.js:104:9)</code></li>
  <li><a href="https://www.khronos.org/registry/webgl/specs/latest/2.0/">https://www.khronos.org/registry/webgl/specs/latest/2.0/</a>: <code>FetchError: request to https://www.khronos.org/registry/webgl/specs/latest/2.0/ failed, reason: getaddrinfo ENOTFOUND www.khronos.org www.khronos.org:443 FetchError: request to https://www.khronos.org/registry/webgl/specs/latest/2.0/ failed, reason: getaddrinfo ENOTFOUND www.khronos.org www.khronos.org:443 at ClientRequest.&lt;anonymous&gt; (/home/dom/github.com/reffy/node_modules/node-fetch/index.js:133:11) at emitOne (events.js:96:13) at ClientRequest.emit (events.js:188:7) at TLSSocket.socketErrorListener (_http_client.js:309:9) at emitOne (events.js:96:13) at TLSSocket.emit (events.js:188:7) at connectErrorNT (net.js:1021:8) at _combinedTickCallback (internal/process/next_tick.js:80:11) at process._tickCallback (internal/process/next_tick.js:104:9)</code></li>
  <li><a href="https://xhr.spec.whatwg.org/">https://xhr.spec.whatwg.org/</a>: <code>FetchError: request to https://xhr.spec.whatwg.org/ failed, reason: getaddrinfo ENOTFOUND xhr.spec.whatwg.org xhr.spec.whatwg.org:443 FetchError: request to https://xhr.spec.whatwg.org/ failed, reason: getaddrinfo ENOTFOUND xhr.spec.whatwg.org xhr.spec.whatwg.org:443 at ClientRequest.&lt;anonymous&gt; (/home/dom/github.com/reffy/node_modules/node-fetch/index.js:133:11) at emitOne (events.js:96:13) at ClientRequest.emit (events.js:188:7) at TLSSocket.socketErrorListener (_http_client.js:309:9) at emitOne (events.js:96:13) at TLSSocket.emit (events.js:188:7) at connectErrorNT (net.js:1021:8) at _combinedTickCallback (internal/process/next_tick.js:80:11) at process._tickCallback (internal/process/next_tick.js:104:9)</code></li>
  <li><a href="https://www.w3.org/TR/image-capture">image-capture</a>: <code>FetchError: request to https://api.w3.org/specifications/image-capture failed, reason: getaddrinfo ENOTFOUND api.w3.org api.w3.org:443 FetchError: request to https://api.w3.org/specifications/image-capture failed, reason: getaddrinfo ENOTFOUND api.w3.org api.w3.org:443 at ClientRequest.&lt;anonymous&gt; (/home/dom/github.com/reffy/node_modules/node-fetch/index.js:133:11) at emitOne (events.js:96:13) at ClientRequest.emit (events.js:188:7) at TLSSocket.socketErrorListener (_http_client.js:309:9) at emitOne (events.js:96:13) at TLSSocket.emit (events.js:188:7) at connectErrorNT (net.js:1021:8) at _combinedTickCallback (internal/process/next_tick.js:80:11) at process._tickCallback (internal/process/next_tick.js:104:9)</code></li>
  <li><a href="https://www.w3.org/TR/IndexedDB-2">IndexedDB-2</a>: <code>FetchError: request to https://api.w3.org/specifications/IndexedDB-2 failed, reason: getaddrinfo ENOTFOUND api.w3.org api.w3.org:443 FetchError: request to https://api.w3.org/specifications/IndexedDB-2 failed, reason: getaddrinfo ENOTFOUND api.w3.org api.w3.org:443 at ClientRequest.&lt;anonymous&gt; (/home/dom/github.com/reffy/node_modules/node-fetch/index.js:133:11) at emitOne (events.js:96:13) at ClientRequest.emit (events.js:188:7) at TLSSocket.socketErrorListener (_http_client.js:309:9) at emitOne (events.js:96:13) at TLSSocket.emit (events.js:188:7) at connectErrorNT (net.js:1021:8) at _combinedTickCallback (internal/process/next_tick.js:80:11) at process._tickCallback (internal/process/next_tick.js:104:9)</code></li>
  <li><a href="https://www.w3.org/TR/input-events-2">input-events-2</a>: <code>FetchError: request to https://api.w3.org/specifications/input-events-2 failed, reason: getaddrinfo ENOTFOUND api.w3.org api.w3.org:443 FetchError: request to https://api.w3.org/specifications/input-events-2 failed, reason: getaddrinfo ENOTFOUND api.w3.org api.w3.org:443 at ClientRequest.&lt;anonymous&gt; (/home/dom/github.com/reffy/node_modules/node-fetch/index.js:133:11) at emitOne (events.js:96:13) at ClientRequest.emit (events.js:188:7) at TLSSocket.socketErrorListener (_http_client.js:309:9) at emitOne (events.js:96:13) at TLSSocket.emit (events.js:188:7) at connectErrorNT (net.js:1021:8) at _combinedTickCallback (internal/process/next_tick.js:80:11) at process._tickCallback (internal/process/next_tick.js:104:9)</code></li>
  <li><a href="https://www.w3.org/TR/magnetometer">magnetometer</a>: <code>FetchError: request to https://api.w3.org/specifications/magnetometer failed, reason: getaddrinfo ENOTFOUND api.w3.org api.w3.org:443 FetchError: request to https://api.w3.org/specifications/magnetometer failed, reason: getaddrinfo ENOTFOUND api.w3.org api.w3.org:443 at ClientRequest.&lt;anonymous&gt; (/home/dom/github.com/reffy/node_modules/node-fetch/index.js:133:11) at emitOne (events.js:96:13) at ClientRequest.emit (events.js:188:7) at TLSSocket.socketErrorListener (_http_client.js:309:9) at emitOne (events.js:96:13) at TLSSocket.emit (events.js:188:7) at connectErrorNT (net.js:1021:8) at _combinedTickCallback (internal/process/next_tick.js:80:11) at process._tickCallback (internal/process/next_tick.js:104:9)</code></li>
  <li><a href="https://www.w3.org/TR/media-source">media-source</a>: <code>FetchError: request to https://api.w3.org/specifications/media-source failed, reason: getaddrinfo ENOTFOUND api.w3.org api.w3.org:443 FetchError: request to https://api.w3.org/specifications/media-source failed, reason: getaddrinfo ENOTFOUND api.w3.org api.w3.org:443 at ClientRequest.&lt;anonymous&gt; (/home/dom/github.com/reffy/node_modules/node-fetch/index.js:133:11) at emitOne (events.js:96:13) at ClientRequest.emit (events.js:188:7) at TLSSocket.socketErrorListener (_http_client.js:309:9) at emitOne (events.js:96:13) at TLSSocket.emit (events.js:188:7) at connectErrorNT (net.js:1021:8) at _combinedTickCallback (internal/process/next_tick.js:80:11) at process._tickCallback (internal/process/next_tick.js:104:9)</code></li>
  <li><a href="https://www.w3.org/TR/mediacapture-depth">mediacapture-depth</a>: <code>FetchError: request to https://api.w3.org/specifications/mediacapture-depth failed, reason: getaddrinfo ENOTFOUND api.w3.org api.w3.org:443 FetchError: request to https://api.w3.org/specifications/mediacapture-depth failed, reason: getaddrinfo ENOTFOUND api.w3.org api.w3.org:443 at ClientRequest.&lt;anonymous&gt; (/home/dom/github.com/reffy/node_modules/node-fetch/index.js:133:11) at emitOne (events.js:96:13) at ClientRequest.emit (events.js:188:7) at TLSSocket.socketErrorListener (_http_client.js:309:9) at emitOne (events.js:96:13) at TLSSocket.emit (events.js:188:7) at connectErrorNT (net.js:1021:8) at _combinedTickCallback (internal/process/next_tick.js:80:11) at process._tickCallback (internal/process/next_tick.js:104:9)</code></li>
  <li><a href="https://www.w3.org/TR/mediacapture-fromelement">mediacapture-fromelement</a>: <code>FetchError: request to https://api.w3.org/specifications/mediacapture-fromelement failed, reason: getaddrinfo ENOTFOUND api.w3.org api.w3.org:443 FetchError: request to https://api.w3.org/specifications/mediacapture-fromelement failed, reason: getaddrinfo ENOTFOUND api.w3.org api.w3.org:443 at ClientRequest.&lt;anonymous&gt; (/home/dom/github.com/reffy/node_modules/node-fetch/index.js:133:11) at emitOne (events.js:96:13) at ClientRequest.emit (events.js:188:7) at TLSSocket.socketErrorListener (_http_client.js:309:9) at emitOne (events.js:96:13) at TLSSocket.emit (events.js:188:7) at connectErrorNT (net.js:1021:8) at _combinedTickCallback (internal/process/next_tick.js:80:11) at process._tickCallback (internal/process/next_tick.js:104:9)</code></li>
  <li><a href="https://www.w3.org/TR/mediacapture-streams">mediacapture-streams</a>: <code>FetchError: request to https://api.w3.org/specifications/mediacapture-streams failed, reason: getaddrinfo ENOTFOUND api.w3.org api.w3.org:443 FetchError: request to https://api.w3.org/specifications/mediacapture-streams failed, reason: getaddrinfo ENOTFOUND api.w3.org api.w3.org:443 at ClientRequest.&lt;anonymous&gt; (/home/dom/github.com/reffy/node_modules/node-fetch/index.js:133:11) at emitOne (events.js:96:13) at ClientRequest.emit (events.js:188:7) at TLSSocket.socketErrorListener (_http_client.js:309:9) at emitOne (events.js:96:13) at TLSSocket.emit (events.js:188:7) at connectErrorNT (net.js:1021:8) at _combinedTickCallback (internal/process/next_tick.js:80:11) at process._tickCallback (internal/process/next_tick.js:104:9)</code></li>
  <li><a href="https://www.w3.org/TR/mediaont-api-1.0">mediaont-api-1.0</a>: <code>FetchError: request to https://api.w3.org/specifications/mediaont-api-1.0 failed, reason: getaddrinfo ENOTFOUND api.w3.org api.w3.org:443 FetchError: request to https://api.w3.org/specifications/mediaont-api-1.0 failed, reason: getaddrinfo ENOTFOUND api.w3.org api.w3.org:443 at ClientRequest.&lt;anonymous&gt; (/home/dom/github.com/reffy/node_modules/node-fetch/index.js:133:11) at emitOne (events.js:96:13) at ClientRequest.emit (events.js:188:7) at TLSSocket.socketErrorListener (_http_client.js:309:9) at emitOne (events.js:96:13) at TLSSocket.emit (events.js:188:7) at connectErrorNT (net.js:1021:8) at _combinedTickCallback (internal/process/next_tick.js:80:11) at process._tickCallback (internal/process/next_tick.js:104:9)</code></li>
  <li><a href="https://www.w3.org/TR/mediastream-recording">mediastream-recording</a>: <code>FetchError: request to https://api.w3.org/specifications/mediastream-recording failed, reason: getaddrinfo ENOTFOUND api.w3.org api.w3.org:443 FetchError: request to https://api.w3.org/specifications/mediastream-recording failed, reason: getaddrinfo ENOTFOUND api.w3.org api.w3.org:443 at ClientRequest.&lt;anonymous&gt; (/home/dom/github.com/reffy/node_modules/node-fetch/index.js:133:11) at emitOne (events.js:96:13) at ClientRequest.emit (events.js:188:7) at TLSSocket.socketErrorListener (_http_client.js:309:9) at emitOne (events.js:96:13) at TLSSocket.emit (events.js:188:7) at connectErrorNT (net.js:1021:8) at _combinedTickCallback (internal/process/next_tick.js:80:11) at process._tickCallback (internal/process/next_tick.js:104:9)</code></li>
  <li><a href="https://www.w3.org/TR/mixed-content">mixed-content</a>: <code>FetchError: request to https://api.w3.org/specifications/mixed-content failed, reason: getaddrinfo ENOTFOUND api.w3.org api.w3.org:443 FetchError: request to https://api.w3.org/specifications/mixed-content failed, reason: getaddrinfo ENOTFOUND api.w3.org api.w3.org:443 at ClientRequest.&lt;anonymous&gt; (/home/dom/github.com/reffy/node_modules/node-fetch/index.js:133:11) at emitOne (events.js:96:13) at ClientRequest.emit (events.js:188:7) at TLSSocket.socketErrorListener (_http_client.js:309:9) at emitOne (events.js:96:13) at TLSSocket.emit (events.js:188:7) at connectErrorNT (net.js:1021:8) at _combinedTickCallback (internal/process/next_tick.js:80:11) at process._tickCallback (internal/process/next_tick.js:104:9)</code></li>
  <li><a href="https://www.w3.org/TR/navigation-timing-2">navigation-timing-2</a>: <code>FetchError: request to https://api.w3.org/specifications/navigation-timing-2 failed, reason: getaddrinfo ENOTFOUND api.w3.org api.w3.org:443 FetchError: request to https://api.w3.org/specifications/navigation-timing-2 failed, reason: getaddrinfo ENOTFOUND api.w3.org api.w3.org:443 at ClientRequest.&lt;anonymous&gt; (/home/dom/github.com/reffy/node_modules/node-fetch/index.js:133:11) at emitOne (events.js:96:13) at ClientRequest.emit (events.js:188:7) at TLSSocket.socketErrorListener (_http_client.js:309:9) at emitOne (events.js:96:13) at TLSSocket.emit (events.js:188:7) at connectErrorNT (net.js:1021:8) at _combinedTickCallback (internal/process/next_tick.js:80:11) at process._tickCallback (internal/process/next_tick.js:104:9)</code></li>
  <li><a href="https://www.w3.org/TR/notifications">notifications</a>: <code>FetchError: request to https://api.w3.org/specifications/notifications failed, reason: getaddrinfo ENOTFOUND api.w3.org api.w3.org:443 FetchError: request to https://api.w3.org/specifications/notifications failed, reason: getaddrinfo ENOTFOUND api.w3.org api.w3.org:443 at ClientRequest.&lt;anonymous&gt; (/home/dom/github.com/reffy/node_modules/node-fetch/index.js:133:11) at emitOne (events.js:96:13) at ClientRequest.emit (events.js:188:7) at TLSSocket.socketErrorListener (_http_client.js:309:9) at emitOne (events.js:96:13) at TLSSocket.emit (events.js:188:7) at connectErrorNT (net.js:1021:8) at _combinedTickCallback (internal/process/next_tick.js:80:11) at process._tickCallback (internal/process/next_tick.js:104:9)</code></li>
  <li><a href="https://www.w3.org/TR/orientation-event">orientation-event</a>: <code>FetchError: request to https://api.w3.org/specifications/orientation-event failed, reason: getaddrinfo ENOTFOUND api.w3.org api.w3.org:443 FetchError: request to https://api.w3.org/specifications/orientation-event failed, reason: getaddrinfo ENOTFOUND api.w3.org api.w3.org:443 at ClientRequest.&lt;anonymous&gt; (/home/dom/github.com/reffy/node_modules/node-fetch/index.js:133:11) at emitOne (events.js:96:13) at ClientRequest.emit (events.js:188:7) at TLSSocket.socketErrorListener (_http_client.js:309:9) at emitOne (events.js:96:13) at TLSSocket.emit (events.js:188:7) at connectErrorNT (net.js:1021:8) at _combinedTickCallback (internal/process/next_tick.js:80:11) at process._tickCallback (internal/process/next_tick.js:104:9)</code></li>
  <li><a href="https://www.w3.org/TR/orientation-sensor">orientation-sensor</a>: <code>FetchError: request to https://api.w3.org/specifications/orientation-sensor failed, reason: getaddrinfo ENOTFOUND api.w3.org api.w3.org:443 FetchError: request to https://api.w3.org/specifications/orientation-sensor failed, reason: getaddrinfo ENOTFOUND api.w3.org api.w3.org:443 at ClientRequest.&lt;anonymous&gt; (/home/dom/github.com/reffy/node_modules/node-fetch/index.js:133:11) at emitOne (events.js:96:13) at ClientRequest.emit (events.js:188:7) at TLSSocket.socketErrorListener (_http_client.js:309:9) at emitOne (events.js:96:13) at TLSSocket.emit (events.js:188:7) at connectErrorNT (net.js:1021:8) at _combinedTickCallback (internal/process/next_tick.js:80:11) at process._tickCallback (internal/process/next_tick.js:104:9)</code></li>
  <li><a href="https://www.w3.org/TR/page-visibility-2">page-visibility-2</a>: <code>FetchError: request to https://api.w3.org/specifications/page-visibility-2 failed, reason: getaddrinfo ENOTFOUND api.w3.org api.w3.org:443 FetchError: request to https://api.w3.org/specifications/page-visibility-2 failed, reason: getaddrinfo ENOTFOUND api.w3.org api.w3.org:443 at ClientRequest.&lt;anonymous&gt; (/home/dom/github.com/reffy/node_modules/node-fetch/index.js:133:11) at emitOne (events.js:96:13) at ClientRequest.emit (events.js:188:7) at TLSSocket.socketErrorListener (_http_client.js:309:9) at emitOne (events.js:96:13) at TLSSocket.emit (events.js:188:7) at connectErrorNT (net.js:1021:8) at _combinedTickCallback (internal/process/next_tick.js:80:11) at process._tickCallback (internal/process/next_tick.js:104:9)</code></li>
  <li><a href="https://www.w3.org/TR/payment-handler">payment-handler</a>: <code>FetchError: request to https://api.w3.org/specifications/payment-handler failed, reason: getaddrinfo ENOTFOUND api.w3.org api.w3.org:443 FetchError: request to https://api.w3.org/specifications/payment-handler failed, reason: getaddrinfo ENOTFOUND api.w3.org api.w3.org:443 at ClientRequest.&lt;anonymous&gt; (/home/dom/github.com/reffy/node_modules/node-fetch/index.js:133:11) at emitOne (events.js:96:13) at ClientRequest.emit (events.js:188:7) at TLSSocket.socketErrorListener (_http_client.js:309:9) at emitOne (events.js:96:13) at TLSSocket.emit (events.js:188:7) at connectErrorNT (net.js:1021:8) at _combinedTickCallback (internal/process/next_tick.js:80:11) at process._tickCallback (internal/process/next_tick.js:104:9)</code></li>
  <li><a href="https://www.w3.org/TR/payment-method-basic-card">payment-method-basic-card</a>: <code>FetchError: request to https://api.w3.org/specifications/payment-method-basic-card failed, reason: getaddrinfo ENOTFOUND api.w3.org api.w3.org:443 FetchError: request to https://api.w3.org/specifications/payment-method-basic-card failed, reason: getaddrinfo ENOTFOUND api.w3.org api.w3.org:443 at ClientRequest.&lt;anonymous&gt; (/home/dom/github.com/reffy/node_modules/node-fetch/index.js:133:11) at emitOne (events.js:96:13) at ClientRequest.emit (events.js:188:7) at TLSSocket.socketErrorListener (_http_client.js:309:9) at emitOne (events.js:96:13) at TLSSocket.emit (events.js:188:7) at connectErrorNT (net.js:1021:8) at _combinedTickCallback (internal/process/next_tick.js:80:11) at process._tickCallback (internal/process/next_tick.js:104:9)</code></li>
  <li><a href="https://www.w3.org/TR/payment-request">payment-request</a>: <code>FetchError: request to https://api.w3.org/specifications/payment-request failed, reason: getaddrinfo ENOTFOUND api.w3.org api.w3.org:443 FetchError: request to https://api.w3.org/specifications/payment-request failed, reason: getaddrinfo ENOTFOUND api.w3.org api.w3.org:443 at ClientRequest.&lt;anonymous&gt; (/home/dom/github.com/reffy/node_modules/node-fetch/index.js:133:11) at emitOne (events.js:96:13) at ClientRequest.emit (events.js:188:7) at TLSSocket.socketErrorListener (_http_client.js:309:9) at emitOne (events.js:96:13) at TLSSocket.emit (events.js:188:7) at connectErrorNT (net.js:1021:8) at _combinedTickCallback (internal/process/next_tick.js:80:11) at process._tickCallback (internal/process/next_tick.js:104:9)</code></li>
  <li><a href="https://www.w3.org/TR/performance-timeline-2">performance-timeline-2</a>: <code>FetchError: request to https://api.w3.org/specifications/performance-timeline-2 failed, reason: getaddrinfo ENOTFOUND api.w3.org api.w3.org:443 FetchError: request to https://api.w3.org/specifications/performance-timeline-2 failed, reason: getaddrinfo ENOTFOUND api.w3.org api.w3.org:443 at ClientRequest.&lt;anonymous&gt; (/home/dom/github.com/reffy/node_modules/node-fetch/index.js:133:11) at emitOne (events.js:96:13) at ClientRequest.emit (events.js:188:7) at TLSSocket.socketErrorListener (_http_client.js:309:9) at emitOne (events.js:96:13) at TLSSocket.emit (events.js:188:7) at connectErrorNT (net.js:1021:8) at _combinedTickCallback (internal/process/next_tick.js:80:11) at process._tickCallback (internal/process/next_tick.js:104:9)</code></li>
  <li><a href="https://www.w3.org/TR/permissions">permissions</a>: <code>FetchError: request to https://api.w3.org/specifications/permissions failed, reason: getaddrinfo ENOTFOUND api.w3.org api.w3.org:443 FetchError: request to https://api.w3.org/specifications/permissions failed, reason: getaddrinfo ENOTFOUND api.w3.org api.w3.org:443 at ClientRequest.&lt;anonymous&gt; (/home/dom/github.com/reffy/node_modules/node-fetch/index.js:133:11) at emitOne (events.js:96:13) at ClientRequest.emit (events.js:188:7) at TLSSocket.socketErrorListener (_http_client.js:309:9) at emitOne (events.js:96:13) at TLSSocket.emit (events.js:188:7) at connectErrorNT (net.js:1021:8) at _combinedTickCallback (internal/process/next_tick.js:80:11) at process._tickCallback (internal/process/next_tick.js:104:9)</code></li>
  <li><a href="https://www.w3.org/TR/pointerevents2">pointerevents2</a>: <code>FetchError: request to https://api.w3.org/specifications/pointerevents2 failed, reason: getaddrinfo ENOTFOUND api.w3.org api.w3.org:443 FetchError: request to https://api.w3.org/specifications/pointerevents2 failed, reason: getaddrinfo ENOTFOUND api.w3.org api.w3.org:443 at ClientRequest.&lt;anonymous&gt; (/home/dom/github.com/reffy/node_modules/node-fetch/index.js:133:11) at emitOne (events.js:96:13) at ClientRequest.emit (events.js:188:7) at TLSSocket.socketErrorListener (_http_client.js:309:9) at emitOne (events.js:96:13) at TLSSocket.emit (events.js:188:7) at connectErrorNT (net.js:1021:8) at _combinedTickCallback (internal/process/next_tick.js:80:11) at process._tickCallback (internal/process/next_tick.js:104:9)</code></li>
  <li><a href="https://www.w3.org/TR/pointerlock-2">pointerlock-2</a>: <code>FetchError: request to https://api.w3.org/specifications/pointerlock-2 failed, reason: getaddrinfo ENOTFOUND api.w3.org api.w3.org:443 FetchError: request to https://api.w3.org/specifications/pointerlock-2 failed, reason: getaddrinfo ENOTFOUND api.w3.org api.w3.org:443 at ClientRequest.&lt;anonymous&gt; (/home/dom/github.com/reffy/node_modules/node-fetch/index.js:133:11) at emitOne (events.js:96:13) at ClientRequest.emit (events.js:188:7) at TLSSocket.socketErrorListener (_http_client.js:309:9) at emitOne (events.js:96:13) at TLSSocket.emit (events.js:188:7) at connectErrorNT (net.js:1021:8) at _combinedTickCallback (internal/process/next_tick.js:80:11) at process._tickCallback (internal/process/next_tick.js:104:9)</code></li>
  <li><a href="https://www.w3.org/TR/preload">preload</a>: <code>FetchError: request to https://api.w3.org/specifications/preload failed, reason: getaddrinfo ENOTFOUND api.w3.org api.w3.org:443 FetchError: request to https://api.w3.org/specifications/preload failed, reason: getaddrinfo ENOTFOUND api.w3.org api.w3.org:443 at ClientRequest.&lt;anonymous&gt; (/home/dom/github.com/reffy/node_modules/node-fetch/index.js:133:11) at emitOne (events.js:96:13) at ClientRequest.emit (events.js:188:7) at TLSSocket.socketErrorListener (_http_client.js:309:9) at emitOne (events.js:96:13) at TLSSocket.emit (events.js:188:7) at connectErrorNT (net.js:1021:8) at _combinedTickCallback (internal/process/next_tick.js:80:11) at process._tickCallback (internal/process/next_tick.js:104:9)</code></li>
  <li><a href="https://www.w3.org/TR/presentation-api">presentation-api</a>: <code>FetchError: request to https://api.w3.org/specifications/presentation-api failed, reason: getaddrinfo ENOTFOUND api.w3.org api.w3.org:443 FetchError: request to https://api.w3.org/specifications/presentation-api failed, reason: getaddrinfo ENOTFOUND api.w3.org api.w3.org:443 at ClientRequest.&lt;anonymous&gt; (/home/dom/github.com/reffy/node_modules/node-fetch/index.js:133:11) at emitOne (events.js:96:13) at ClientRequest.emit (events.js:188:7) at TLSSocket.socketErrorListener (_http_client.js:309:9) at emitOne (events.js:96:13) at TLSSocket.emit (events.js:188:7) at connectErrorNT (net.js:1021:8) at _combinedTickCallback (internal/process/next_tick.js:80:11) at process._tickCallback (internal/process/next_tick.js:104:9)</code></li>
  <li><a href="https://www.w3.org/TR/progress-events">progress-events</a>: <code>FetchError: request to https://api.w3.org/specifications/progress-events failed, reason: getaddrinfo ENOTFOUND api.w3.org api.w3.org:443 FetchError: request to https://api.w3.org/specifications/progress-events failed, reason: getaddrinfo ENOTFOUND api.w3.org api.w3.org:443 at ClientRequest.&lt;anonymous&gt; (/home/dom/github.com/reffy/node_modules/node-fetch/index.js:133:11) at emitOne (events.js:96:13) at ClientRequest.emit (events.js:188:7) at TLSSocket.socketErrorListener (_http_client.js:309:9) at emitOne (events.js:96:13) at TLSSocket.emit (events.js:188:7) at connectErrorNT (net.js:1021:8) at _combinedTickCallback (internal/process/next_tick.js:80:11) at process._tickCallback (internal/process/next_tick.js:104:9)</code></li>
  <li><a href="https://www.w3.org/TR/proximity">proximity</a>: <code>FetchError: request to https://api.w3.org/specifications/proximity failed, reason: getaddrinfo ENOTFOUND api.w3.org api.w3.org:443 FetchError: request to https://api.w3.org/specifications/proximity failed, reason: getaddrinfo ENOTFOUND api.w3.org api.w3.org:443 at ClientRequest.&lt;anonymous&gt; (/home/dom/github.com/reffy/node_modules/node-fetch/index.js:133:11) at emitOne (events.js:96:13) at ClientRequest.emit (events.js:188:7) at TLSSocket.socketErrorListener (_http_client.js:309:9) at emitOne (events.js:96:13) at TLSSocket.emit (events.js:188:7) at connectErrorNT (net.js:1021:8) at _combinedTickCallback (internal/process/next_tick.js:80:11) at process._tickCallback (internal/process/next_tick.js:104:9)</code></li>
  <li><a href="https://www.w3.org/TR/push-api">push-api</a>: <code>FetchError: request to https://api.w3.org/specifications/push-api failed, reason: getaddrinfo ENOTFOUND api.w3.org api.w3.org:443 FetchError: request to https://api.w3.org/specifications/push-api failed, reason: getaddrinfo ENOTFOUND api.w3.org api.w3.org:443 at ClientRequest.&lt;anonymous&gt; (/home/dom/github.com/reffy/node_modules/node-fetch/index.js:133:11) at emitOne (events.js:96:13) at ClientRequest.emit (events.js:188:7) at TLSSocket.socketErrorListener (_http_client.js:309:9) at emitOne (events.js:96:13) at TLSSocket.emit (events.js:188:7) at connectErrorNT (net.js:1021:8) at _combinedTickCallback (internal/process/next_tick.js:80:11) at process._tickCallback (internal/process/next_tick.js:104:9)</code></li>
  <li><a href="https://www.w3.org/TR/referrer-policy">referrer-policy</a>: <code>FetchError: request to https://api.w3.org/specifications/referrer-policy failed, reason: getaddrinfo ENOTFOUND api.w3.org api.w3.org:443 FetchError: request to https://api.w3.org/specifications/referrer-policy failed, reason: getaddrinfo ENOTFOUND api.w3.org api.w3.org:443 at ClientRequest.&lt;anonymous&gt; (/home/dom/github.com/reffy/node_modules/node-fetch/index.js:133:11) at emitOne (events.js:96:13) at ClientRequest.emit (events.js:188:7) at TLSSocket.socketErrorListener (_http_client.js:309:9) at emitOne (events.js:96:13) at TLSSocket.emit (events.js:188:7) at connectErrorNT (net.js:1021:8) at _combinedTickCallback (internal/process/next_tick.js:80:11) at process._tickCallback (internal/process/next_tick.js:104:9)</code></li>
  <li><a href="https://www.w3.org/TR/remote-playback">remote-playback</a>: <code>FetchError: request to https://api.w3.org/specifications/remote-playback failed, reason: getaddrinfo ENOTFOUND api.w3.org api.w3.org:443 FetchError: request to https://api.w3.org/specifications/remote-playback failed, reason: getaddrinfo ENOTFOUND api.w3.org api.w3.org:443 at ClientRequest.&lt;anonymous&gt; (/home/dom/github.com/reffy/node_modules/node-fetch/index.js:133:11) at emitOne (events.js:96:13) at ClientRequest.emit (events.js:188:7) at TLSSocket.socketErrorListener (_http_client.js:309:9) at emitOne (events.js:96:13) at TLSSocket.emit (events.js:188:7) at connectErrorNT (net.js:1021:8) at _combinedTickCallback (internal/process/next_tick.js:80:11) at process._tickCallback (internal/process/next_tick.js:104:9)</code></li>
  <li><a href="https://www.w3.org/TR/requestidlecallback">requestidlecallback</a>: <code>FetchError: request to https://api.w3.org/specifications/requestidlecallback failed, reason: getaddrinfo ENOTFOUND api.w3.org api.w3.org:443 FetchError: request to https://api.w3.org/specifications/requestidlecallback failed, reason: getaddrinfo ENOTFOUND api.w3.org api.w3.org:443 at ClientRequest.&lt;anonymous&gt; (/home/dom/github.com/reffy/node_modules/node-fetch/index.js:133:11) at emitOne (events.js:96:13) at ClientRequest.emit (events.js:188:7) at TLSSocket.socketErrorListener (_http_client.js:309:9) at emitOne (events.js:96:13) at TLSSocket.emit (events.js:188:7) at connectErrorNT (net.js:1021:8) at _combinedTickCallback (internal/process/next_tick.js:80:11) at process._tickCallback (internal/process/next_tick.js:104:9)</code></li>
  <li><a href="https://www.w3.org/TR/resource-timing-2">resource-timing-2</a>: <code>FetchError: request to https://api.w3.org/specifications/resource-timing-2 failed, reason: getaddrinfo ENOTFOUND api.w3.org api.w3.org:443 FetchError: request to https://api.w3.org/specifications/resource-timing-2 failed, reason: getaddrinfo ENOTFOUND api.w3.org api.w3.org:443 at ClientRequest.&lt;anonymous&gt; (/home/dom/github.com/reffy/node_modules/node-fetch/index.js:133:11) at emitOne (events.js:96:13) at ClientRequest.emit (events.js:188:7) at TLSSocket.socketErrorListener (_http_client.js:309:9) at emitOne (events.js:96:13) at TLSSocket.emit (events.js:188:7) at connectErrorNT (net.js:1021:8) at _combinedTickCallback (internal/process/next_tick.js:80:11) at process._tickCallback (internal/process/next_tick.js:104:9)</code></li>
  <li><a href="https://www.w3.org/TR/screen-capture">screen-capture</a>: <code>FetchError: request to https://api.w3.org/specifications/screen-capture failed, reason: getaddrinfo ENOTFOUND api.w3.org api.w3.org:443 FetchError: request to https://api.w3.org/specifications/screen-capture failed, reason: getaddrinfo ENOTFOUND api.w3.org api.w3.org:443 at ClientRequest.&lt;anonymous&gt; (/home/dom/github.com/reffy/node_modules/node-fetch/index.js:133:11) at emitOne (events.js:96:13) at ClientRequest.emit (events.js:188:7) at TLSSocket.socketErrorListener (_http_client.js:309:9) at emitOne (events.js:96:13) at TLSSocket.emit (events.js:188:7) at connectErrorNT (net.js:1021:8) at _combinedTickCallback (internal/process/next_tick.js:80:11) at process._tickCallback (internal/process/next_tick.js:104:9)</code></li>
  <li><a href="https://www.w3.org/TR/screen-orientation">screen-orientation</a>: <code>FetchError: request to https://api.w3.org/specifications/screen-orientation failed, reason: getaddrinfo ENOTFOUND api.w3.org api.w3.org:443 FetchError: request to https://api.w3.org/specifications/screen-orientation failed, reason: getaddrinfo ENOTFOUND api.w3.org api.w3.org:443 at ClientRequest.&lt;anonymous&gt; (/home/dom/github.com/reffy/node_modules/node-fetch/index.js:133:11) at emitOne (events.js:96:13) at ClientRequest.emit (events.js:188:7) at TLSSocket.socketErrorListener (_http_client.js:309:9) at emitOne (events.js:96:13) at TLSSocket.emit (events.js:188:7) at connectErrorNT (net.js:1021:8) at _combinedTickCallback (internal/process/next_tick.js:80:11) at process._tickCallback (internal/process/next_tick.js:104:9)</code></li>
  <li><a href="https://www.w3.org/TR/secure-contexts">secure-contexts</a>: <code>FetchError: request to https://api.w3.org/specifications/secure-contexts failed, reason: getaddrinfo ENOTFOUND api.w3.org api.w3.org:443 FetchError: request to https://api.w3.org/specifications/secure-contexts failed, reason: getaddrinfo ENOTFOUND api.w3.org api.w3.org:443 at ClientRequest.&lt;anonymous&gt; (/home/dom/github.com/reffy/node_modules/node-fetch/index.js:133:11) at emitOne (events.js:96:13) at ClientRequest.emit (events.js:188:7) at TLSSocket.socketErrorListener (_http_client.js:309:9) at emitOne (events.js:96:13) at TLSSocket.emit (events.js:188:7) at connectErrorNT (net.js:1021:8) at _combinedTickCallback (internal/process/next_tick.js:80:11) at process._tickCallback (internal/process/next_tick.js:104:9)</code></li>
  <li><a href="https://www.w3.org/TR/selection-api">selection-api</a>: <code>FetchError: request to https://api.w3.org/specifications/selection-api failed, reason: getaddrinfo ENOTFOUND api.w3.org api.w3.org:443 FetchError: request to https://api.w3.org/specifications/selection-api failed, reason: getaddrinfo ENOTFOUND api.w3.org api.w3.org:443 at ClientRequest.&lt;anonymous&gt; (/home/dom/github.com/reffy/node_modules/node-fetch/index.js:133:11) at emitOne (events.js:96:13) at ClientRequest.emit (events.js:188:7) at TLSSocket.socketErrorListener (_http_client.js:309:9) at emitOne (events.js:96:13) at TLSSocket.emit (events.js:188:7) at connectErrorNT (net.js:1021:8) at _combinedTickCallback (internal/process/next_tick.js:80:11) at process._tickCallback (internal/process/next_tick.js:104:9)</code></li>
  <li><a href="https://www.w3.org/TR/selectors-api">selectors-api</a>: <code>FetchError: request to https://api.w3.org/specifications/selectors-api failed, reason: getaddrinfo ENOTFOUND api.w3.org api.w3.org:443 FetchError: request to https://api.w3.org/specifications/selectors-api failed, reason: getaddrinfo ENOTFOUND api.w3.org api.w3.org:443 at ClientRequest.&lt;anonymous&gt; (/home/dom/github.com/reffy/node_modules/node-fetch/index.js:133:11) at emitOne (events.js:96:13) at ClientRequest.emit (events.js:188:7) at TLSSocket.socketErrorListener (_http_client.js:309:9) at emitOne (events.js:96:13) at TLSSocket.emit (events.js:188:7) at connectErrorNT (net.js:1021:8) at _combinedTickCallback (internal/process/next_tick.js:80:11) at process._tickCallback (internal/process/next_tick.js:104:9)</code></li>
  <li><a href="https://www.w3.org/TR/service-workers-1">service-workers-1</a>: <code>FetchError: request to https://api.w3.org/specifications/service-workers-1 failed, reason: getaddrinfo ENOTFOUND api.w3.org api.w3.org:443 FetchError: request to https://api.w3.org/specifications/service-workers-1 failed, reason: getaddrinfo ENOTFOUND api.w3.org api.w3.org:443 at ClientRequest.&lt;anonymous&gt; (/home/dom/github.com/reffy/node_modules/node-fetch/index.js:133:11) at emitOne (events.js:96:13) at ClientRequest.emit (events.js:188:7) at TLSSocket.socketErrorListener (_http_client.js:309:9) at emitOne (events.js:96:13) at TLSSocket.emit (events.js:188:7) at connectErrorNT (net.js:1021:8) at _combinedTickCallback (internal/process/next_tick.js:80:11) at process._tickCallback (internal/process/next_tick.js:104:9)</code></li>
  <li><a href="https://www.w3.org/TR/shadow-dom">shadow-dom</a>: <code>FetchError: request to https://api.w3.org/specifications/shadow-dom failed, reason: getaddrinfo ENOTFOUND api.w3.org api.w3.org:443 FetchError: request to https://api.w3.org/specifications/shadow-dom failed, reason: getaddrinfo ENOTFOUND api.w3.org api.w3.org:443 at ClientRequest.&lt;anonymous&gt; (/home/dom/github.com/reffy/node_modules/node-fetch/index.js:133:11) at emitOne (events.js:96:13) at ClientRequest.emit (events.js:188:7) at TLSSocket.socketErrorListener (_http_client.js:309:9) at emitOne (events.js:96:13) at TLSSocket.emit (events.js:188:7) at connectErrorNT (net.js:1021:8) at _combinedTickCallback (internal/process/next_tick.js:80:11) at process._tickCallback (internal/process/next_tick.js:104:9)</code></li>
  <li><a href="https://www.w3.org/TR/SVG2">SVG2</a>: <code>FetchError: request to https://api.w3.org/specifications/SVG2 failed, reason: getaddrinfo ENOTFOUND api.w3.org api.w3.org:443 FetchError: request to https://api.w3.org/specifications/SVG2 failed, reason: getaddrinfo ENOTFOUND api.w3.org api.w3.org:443 at ClientRequest.&lt;anonymous&gt; (/home/dom/github.com/reffy/node_modules/node-fetch/index.js:133:11) at emitOne (events.js:96:13) at ClientRequest.emit (events.js:188:7) at TLSSocket.socketErrorListener (_http_client.js:309:9) at emitOne (events.js:96:13) at TLSSocket.emit (events.js:188:7) at connectErrorNT (net.js:1021:8) at _combinedTickCallback (internal/process/next_tick.js:80:11) at process._tickCallback (internal/process/next_tick.js:104:9)</code></li>
  <li><a href="https://www.w3.org/TR/touch-events">touch-events</a>: <code>FetchError: request to https://api.w3.org/specifications/touch-events failed, reason: getaddrinfo ENOTFOUND api.w3.org api.w3.org:443 FetchError: request to https://api.w3.org/specifications/touch-events failed, reason: getaddrinfo ENOTFOUND api.w3.org api.w3.org:443 at ClientRequest.&lt;anonymous&gt; (/home/dom/github.com/reffy/node_modules/node-fetch/index.js:133:11) at emitOne (events.js:96:13) at ClientRequest.emit (events.js:188:7) at TLSSocket.socketErrorListener (_http_client.js:309:9) at emitOne (events.js:96:13) at TLSSocket.emit (events.js:188:7) at connectErrorNT (net.js:1021:8) at _combinedTickCallback (internal/process/next_tick.js:80:11) at process._tickCallback (internal/process/next_tick.js:104:9)</code></li>
  <li><a href="https://www.w3.org/TR/uievents">uievents</a>: <code>FetchError: request to https://api.w3.org/specifications/uievents failed, reason: getaddrinfo ENOTFOUND api.w3.org api.w3.org:443 FetchError: request to https://api.w3.org/specifications/uievents failed, reason: getaddrinfo ENOTFOUND api.w3.org api.w3.org:443 at ClientRequest.&lt;anonymous&gt; (/home/dom/github.com/reffy/node_modules/node-fetch/index.js:133:11) at emitOne (events.js:96:13) at ClientRequest.emit (events.js:188:7) at TLSSocket.socketErrorListener (_http_client.js:309:9) at emitOne (events.js:96:13) at TLSSocket.emit (events.js:188:7) at connectErrorNT (net.js:1021:8) at _combinedTickCallback (internal/process/next_tick.js:80:11) at process._tickCallback (internal/process/next_tick.js:104:9)</code></li>
  <li><a href="https://www.w3.org/TR/uievents-code">uievents-code</a>: <code>FetchError: request to https://api.w3.org/specifications/uievents-code failed, reason: getaddrinfo ENOTFOUND api.w3.org api.w3.org:443 FetchError: request to https://api.w3.org/specifications/uievents-code failed, reason: getaddrinfo ENOTFOUND api.w3.org api.w3.org:443 at ClientRequest.&lt;anonymous&gt; (/home/dom/github.com/reffy/node_modules/node-fetch/index.js:133:11) at emitOne (events.js:96:13) at ClientRequest.emit (events.js:188:7) at TLSSocket.socketErrorListener (_http_client.js:309:9) at emitOne (events.js:96:13) at TLSSocket.emit (events.js:188:7) at connectErrorNT (net.js:1021:8) at _combinedTickCallback (internal/process/next_tick.js:80:11) at process._tickCallback (internal/process/next_tick.js:104:9)</code></li>
  <li><a href="https://www.w3.org/TR/uievents-key">uievents-key</a>: <code>FetchError: request to https://api.w3.org/specifications/uievents-key failed, reason: getaddrinfo ENOTFOUND api.w3.org api.w3.org:443 FetchError: request to https://api.w3.org/specifications/uievents-key failed, reason: getaddrinfo ENOTFOUND api.w3.org api.w3.org:443 at ClientRequest.&lt;anonymous&gt; (/home/dom/github.com/reffy/node_modules/node-fetch/index.js:133:11) at emitOne (events.js:96:13) at ClientRequest.emit (events.js:188:7) at TLSSocket.socketErrorListener (_http_client.js:309:9) at emitOne (events.js:96:13) at TLSSocket.emit (events.js:188:7) at connectErrorNT (net.js:1021:8) at _combinedTickCallback (internal/process/next_tick.js:80:11) at process._tickCallback (internal/process/next_tick.js:104:9)</code></li>
  <li><a href="https://www.w3.org/TR/UISecurity">UISecurity</a>: <code>FetchError: request to https://api.w3.org/specifications/UISecurity failed, reason: getaddrinfo ENOTFOUND api.w3.org api.w3.org:443 FetchError: request to https://api.w3.org/specifications/UISecurity failed, reason: getaddrinfo ENOTFOUND api.w3.org api.w3.org:443 at ClientRequest.&lt;anonymous&gt; (/home/dom/github.com/reffy/node_modules/node-fetch/index.js:133:11) at emitOne (events.js:96:13) at ClientRequest.emit (events.js:188:7) at TLSSocket.socketErrorListener (_http_client.js:309:9) at emitOne (events.js:96:13) at TLSSocket.emit (events.js:188:7) at connectErrorNT (net.js:1021:8) at _combinedTickCallback (internal/process/next_tick.js:80:11) at process._tickCallback (internal/process/next_tick.js:104:9)</code></li>
  <li><a href="https://www.w3.org/TR/user-timing-2">user-timing-2</a>: <code>FetchError: request to https://api.w3.org/specifications/user-timing-2 failed, reason: getaddrinfo ENOTFOUND api.w3.org api.w3.org:443 FetchError: request to https://api.w3.org/specifications/user-timing-2 failed, reason: getaddrinfo ENOTFOUND api.w3.org api.w3.org:443 at ClientRequest.&lt;anonymous&gt; (/home/dom/github.com/reffy/node_modules/node-fetch/index.js:133:11) at emitOne (events.js:96:13) at ClientRequest.emit (events.js:188:7) at TLSSocket.socketErrorListener (_http_client.js:309:9) at emitOne (events.js:96:13) at TLSSocket.emit (events.js:188:7) at connectErrorNT (net.js:1021:8) at _combinedTickCallback (internal/process/next_tick.js:80:11) at process._tickCallback (internal/process/next_tick.js:104:9)</code></li>
  <li><a href="https://www.w3.org/TR/vibration">vibration</a>: <code>FetchError: request to https://api.w3.org/specifications/vibration failed, reason: getaddrinfo ENOTFOUND api.w3.org api.w3.org:443 FetchError: request to https://api.w3.org/specifications/vibration failed, reason: getaddrinfo ENOTFOUND api.w3.org api.w3.org:443 at ClientRequest.&lt;anonymous&gt; (/home/dom/github.com/reffy/node_modules/node-fetch/index.js:133:11) at emitOne (events.js:96:13) at ClientRequest.emit (events.js:188:7) at TLSSocket.socketErrorListener (_http_client.js:309:9) at emitOne (events.js:96:13) at TLSSocket.emit (events.js:188:7) at connectErrorNT (net.js:1021:8) at _combinedTickCallback (internal/process/next_tick.js:80:11) at process._tickCallback (internal/process/next_tick.js:104:9)</code></li>
  <li><a href="https://www.w3.org/TR/wake-lock">wake-lock</a>: <code>FetchError: request to https://api.w3.org/specifications/wake-lock failed, reason: getaddrinfo ENOTFOUND api.w3.org api.w3.org:443 FetchError: request to https://api.w3.org/specifications/wake-lock failed, reason: getaddrinfo ENOTFOUND api.w3.org api.w3.org:443 at ClientRequest.&lt;anonymous&gt; (/home/dom/github.com/reffy/node_modules/node-fetch/index.js:133:11) at emitOne (events.js:96:13) at ClientRequest.emit (events.js:188:7) at TLSSocket.socketErrorListener (_http_client.js:309:9) at emitOne (events.js:96:13) at TLSSocket.emit (events.js:188:7) at connectErrorNT (net.js:1021:8) at _combinedTickCallback (internal/process/next_tick.js:80:11) at process._tickCallback (internal/process/next_tick.js:104:9)</code></li>
  <li><a href="https://www.w3.org/TR/web-animations-1">web-animations-1</a>: <code>FetchError: request to https://api.w3.org/specifications/web-animations-1 failed, reason: getaddrinfo ENOTFOUND api.w3.org api.w3.org:443 FetchError: request to https://api.w3.org/specifications/web-animations-1 failed, reason: getaddrinfo ENOTFOUND api.w3.org api.w3.org:443 at ClientRequest.&lt;anonymous&gt; (/home/dom/github.com/reffy/node_modules/node-fetch/index.js:133:11) at emitOne (events.js:96:13) at ClientRequest.emit (events.js:188:7) at TLSSocket.socketErrorListener (_http_client.js:309:9) at emitOne (events.js:96:13) at TLSSocket.emit (events.js:188:7) at connectErrorNT (net.js:1021:8) at _combinedTickCallback (internal/process/next_tick.js:80:11) at process._tickCallback (internal/process/next_tick.js:104:9)</code></li>
  <li><a href="https://www.w3.org/TR/webaudio">webaudio</a>: <code>FetchError: request to https://api.w3.org/specifications/webaudio failed, reason: getaddrinfo ENOTFOUND api.w3.org api.w3.org:443 FetchError: request to https://api.w3.org/specifications/webaudio failed, reason: getaddrinfo ENOTFOUND api.w3.org api.w3.org:443 at ClientRequest.&lt;anonymous&gt; (/home/dom/github.com/reffy/node_modules/node-fetch/index.js:133:11) at emitOne (events.js:96:13) at ClientRequest.emit (events.js:188:7) at TLSSocket.socketErrorListener (_http_client.js:309:9) at emitOne (events.js:96:13) at TLSSocket.emit (events.js:188:7) at connectErrorNT (net.js:1021:8) at _combinedTickCallback (internal/process/next_tick.js:80:11) at process._tickCallback (internal/process/next_tick.js:104:9)</code></li>
  <li><a href="https://www.w3.org/TR/webauthn">webauthn</a>: <code>FetchError: request to https://api.w3.org/specifications/webauthn failed, reason: getaddrinfo ENOTFOUND api.w3.org api.w3.org:443 FetchError: request to https://api.w3.org/specifications/webauthn failed, reason: getaddrinfo ENOTFOUND api.w3.org api.w3.org:443 at ClientRequest.&lt;anonymous&gt; (/home/dom/github.com/reffy/node_modules/node-fetch/index.js:133:11) at emitOne (events.js:96:13) at ClientRequest.emit (events.js:188:7) at TLSSocket.socketErrorListener (_http_client.js:309:9) at emitOne (events.js:96:13) at TLSSocket.emit (events.js:188:7) at connectErrorNT (net.js:1021:8) at _combinedTickCallback (internal/process/next_tick.js:80:11) at process._tickCallback (internal/process/next_tick.js:104:9)</code></li>
  <li><a href="https://www.w3.org/TR/WebCryptoAPI">WebCryptoAPI</a>: <code>FetchError: request to https://api.w3.org/specifications/WebCryptoAPI failed, reason: getaddrinfo ENOTFOUND api.w3.org api.w3.org:443 FetchError: request to https://api.w3.org/specifications/WebCryptoAPI failed, reason: getaddrinfo ENOTFOUND api.w3.org api.w3.org:443 at ClientRequest.&lt;anonymous&gt; (/home/dom/github.com/reffy/node_modules/node-fetch/index.js:133:11) at emitOne (events.js:96:13) at ClientRequest.emit (events.js:188:7) at TLSSocket.socketErrorListener (_http_client.js:309:9) at emitOne (events.js:96:13) at TLSSocket.emit (events.js:188:7) at connectErrorNT (net.js:1021:8) at _combinedTickCallback (internal/process/next_tick.js:80:11) at process._tickCallback (internal/process/next_tick.js:104:9)</code></li>
  <li><a href="https://www.w3.org/TR/webdriver">webdriver</a>: <code>FetchError: request to https://api.w3.org/specifications/webdriver failed, reason: getaddrinfo ENOTFOUND api.w3.org api.w3.org:443 FetchError: request to https://api.w3.org/specifications/webdriver failed, reason: getaddrinfo ENOTFOUND api.w3.org api.w3.org:443 at ClientRequest.&lt;anonymous&gt; (/home/dom/github.com/reffy/node_modules/node-fetch/index.js:133:11) at emitOne (events.js:96:13) at ClientRequest.emit (events.js:188:7) at TLSSocket.socketErrorListener (_http_client.js:309:9) at emitOne (events.js:96:13) at TLSSocket.emit (events.js:188:7) at connectErrorNT (net.js:1021:8) at _combinedTickCallback (internal/process/next_tick.js:80:11) at process._tickCallback (internal/process/next_tick.js:104:9)</code></li>
  <li><a href="https://www.w3.org/TR/WebIDL-1">WebIDL-1</a>: <code>FetchError: request to https://api.w3.org/specifications/WebIDL-1 failed, reason: getaddrinfo ENOTFOUND api.w3.org api.w3.org:443 FetchError: request to https://api.w3.org/specifications/WebIDL-1 failed, reason: getaddrinfo ENOTFOUND api.w3.org api.w3.org:443 at ClientRequest.&lt;anonymous&gt; (/home/dom/github.com/reffy/node_modules/node-fetch/index.js:133:11) at emitOne (events.js:96:13) at ClientRequest.emit (events.js:188:7) at TLSSocket.socketErrorListener (_http_client.js:309:9) at emitOne (events.js:96:13) at TLSSocket.emit (events.js:188:7) at connectErrorNT (net.js:1021:8) at _combinedTickCallback (internal/process/next_tick.js:80:11) at process._tickCallback (internal/process/next_tick.js:104:9)</code></li>
  <li><a href="https://www.w3.org/TR/webmessaging">webmessaging</a>: <code>FetchError: request to https://api.w3.org/specifications/webmessaging failed, reason: getaddrinfo ENOTFOUND api.w3.org api.w3.org:443 FetchError: request to https://api.w3.org/specifications/webmessaging failed, reason: getaddrinfo ENOTFOUND api.w3.org api.w3.org:443 at ClientRequest.&lt;anonymous&gt; (/home/dom/github.com/reffy/node_modules/node-fetch/index.js:133:11) at emitOne (events.js:96:13) at ClientRequest.emit (events.js:188:7) at TLSSocket.socketErrorListener (_http_client.js:309:9) at emitOne (events.js:96:13) at TLSSocket.emit (events.js:188:7) at connectErrorNT (net.js:1021:8) at _combinedTickCallback (internal/process/next_tick.js:80:11) at process._tickCallback (internal/process/next_tick.js:104:9)</code></li>
  <li><a href="https://www.w3.org/TR/webrtc">webrtc</a>: <code>FetchError: request to https://api.w3.org/specifications/webrtc failed, reason: getaddrinfo ENOTFOUND api.w3.org api.w3.org:443 FetchError: request to https://api.w3.org/specifications/webrtc failed, reason: getaddrinfo ENOTFOUND api.w3.org api.w3.org:443 at ClientRequest.&lt;anonymous&gt; (/home/dom/github.com/reffy/node_modules/node-fetch/index.js:133:11) at emitOne (events.js:96:13) at ClientRequest.emit (events.js:188:7) at TLSSocket.socketErrorListener (_http_client.js:309:9) at emitOne (events.js:96:13) at TLSSocket.emit (events.js:188:7) at connectErrorNT (net.js:1021:8) at _combinedTickCallback (internal/process/next_tick.js:80:11) at process._tickCallback (internal/process/next_tick.js:104:9)</code></li>
  <li><a href="https://www.w3.org/TR/webrtc-stats">webrtc-stats</a>: <code>FetchError: request to https://api.w3.org/specifications/webrtc-stats failed, reason: getaddrinfo ENOTFOUND api.w3.org api.w3.org:443 FetchError: request to https://api.w3.org/specifications/webrtc-stats failed, reason: getaddrinfo ENOTFOUND api.w3.org api.w3.org:443 at ClientRequest.&lt;anonymous&gt; (/home/dom/github.com/reffy/node_modules/node-fetch/index.js:133:11) at emitOne (events.js:96:13) at ClientRequest.emit (events.js:188:7) at TLSSocket.socketErrorListener (_http_client.js:309:9) at emitOne (events.js:96:13) at TLSSocket.emit (events.js:188:7) at connectErrorNT (net.js:1021:8) at _combinedTickCallback (internal/process/next_tick.js:80:11) at process._tickCallback (internal/process/next_tick.js:104:9)</code></li>
  <li><a href="https://www.w3.org/TR/websockets">websockets</a>: <code>FetchError: request to https://api.w3.org/specifications/websockets failed, reason: getaddrinfo ENOTFOUND api.w3.org api.w3.org:443 FetchError: request to https://api.w3.org/specifications/websockets failed, reason: getaddrinfo ENOTFOUND api.w3.org api.w3.org:443 at ClientRequest.&lt;anonymous&gt; (/home/dom/github.com/reffy/node_modules/node-fetch/index.js:133:11) at emitOne (events.js:96:13) at ClientRequest.emit (events.js:188:7) at TLSSocket.socketErrorListener (_http_client.js:309:9) at emitOne (events.js:96:13) at TLSSocket.emit (events.js:188:7) at connectErrorNT (net.js:1021:8) at _combinedTickCallback (internal/process/next_tick.js:80:11) at process._tickCallback (internal/process/next_tick.js:104:9)</code></li>
  <li><a href="https://www.w3.org/TR/webstorage">webstorage</a>: <code>FetchError: request to https://api.w3.org/specifications/webstorage failed, reason: getaddrinfo ENOTFOUND api.w3.org api.w3.org:443 FetchError: request to https://api.w3.org/specifications/webstorage failed, reason: getaddrinfo ENOTFOUND api.w3.org api.w3.org:443 at ClientRequest.&lt;anonymous&gt; (/home/dom/github.com/reffy/node_modules/node-fetch/index.js:133:11) at emitOne (events.js:96:13) at ClientRequest.emit (events.js:188:7) at TLSSocket.socketErrorListener (_http_client.js:309:9) at emitOne (events.js:96:13) at TLSSocket.emit (events.js:188:7) at connectErrorNT (net.js:1021:8) at _combinedTickCallback (internal/process/next_tick.js:80:11) at process._tickCallback (internal/process/next_tick.js:104:9)</code></li>
  <li><a href="https://www.w3.org/TR/webvtt1">webvtt1</a>: <code>FetchError: request to https://api.w3.org/specifications/webvtt1 failed, reason: getaddrinfo ENOTFOUND api.w3.org api.w3.org:443 FetchError: request to https://api.w3.org/specifications/webvtt1 failed, reason: getaddrinfo ENOTFOUND api.w3.org api.w3.org:443 at ClientRequest.&lt;anonymous&gt; (/home/dom/github.com/reffy/node_modules/node-fetch/index.js:133:11) at emitOne (events.js:96:13) at ClientRequest.emit (events.js:188:7) at TLSSocket.socketErrorListener (_http_client.js:309:9) at emitOne (events.js:96:13) at TLSSocket.emit (events.js:188:7) at connectErrorNT (net.js:1021:8) at _combinedTickCallback (internal/process/next_tick.js:80:11) at process._tickCallback (internal/process/next_tick.js:104:9)</code></li>
  <li><a href="https://www.w3.org/TR/workers">workers</a>: <code>FetchError: request to https://api.w3.org/specifications/workers failed, reason: getaddrinfo ENOTFOUND api.w3.org api.w3.org:443 FetchError: request to https://api.w3.org/specifications/workers failed, reason: getaddrinfo ENOTFOUND api.w3.org api.w3.org:443 at ClientRequest.&lt;anonymous&gt; (/home/dom/github.com/reffy/node_modules/node-fetch/index.js:133:11) at emitOne (events.js:96:13) at ClientRequest.emit (events.js:188:7) at TLSSocket.socketErrorListener (_http_client.js:309:9) at emitOne (events.js:96:13) at TLSSocket.emit (events.js:188:7) at connectErrorNT (net.js:1021:8) at _combinedTickCallback (internal/process/next_tick.js:80:11) at process._tickCallback (internal/process/next_tick.js:104:9)</code></li>
  <li><a href="https://www.w3.org/TR/worklets-1">worklets-1</a>: <code>FetchError: request to https://api.w3.org/specifications/worklets-1 failed, reason: getaddrinfo ENOTFOUND api.w3.org api.w3.org:443 FetchError: request to https://api.w3.org/specifications/worklets-1 failed, reason: getaddrinfo ENOTFOUND api.w3.org api.w3.org:443 at ClientRequest.&lt;anonymous&gt; (/home/dom/github.com/reffy/node_modules/node-fetch/index.js:133:11) at emitOne (events.js:96:13) at ClientRequest.emit (events.js:188:7) at TLSSocket.socketErrorListener (_http_client.js:309:9) at emitOne (events.js:96:13) at TLSSocket.emit (events.js:188:7) at connectErrorNT (net.js:1021:8) at _combinedTickCallback (internal/process/next_tick.js:80:11) at process._tickCallback (internal/process/next_tick.js:104:9)</code></li>
  </ul>
  <p>=&gt; 110 specifications found</p>
  </section>
  <section id="specifications-without-normative-dependencies" class="level2">
  <h2>Specifications without normative dependencies</h2>
  <p>=&gt; 0 specification found</p>
  </section>
  <section id="specifications-without-webidl-definitions" class="level2">
  <h2>Specifications without WebIDL definitions</h2>
  <ul>
  <li><a href="https://w3c.github.io/editing/contentEditable.html">ContentEditable</a></li>
  </ul>
  <p>=&gt; 1 specification found</p>
  <p>Not all specifications define IDL content, presence in this list may be completely normal. Reffy's current focus is on IDL specifications, the number of specifications listed here should remain minimal.</p>
  </section>
  <section id="list-of-specifications-with-invalid-webidl-content" class="level2">
  <h2>List of specifications with invalid WebIDL content</h2>
  <p>=&gt; 0 specification found</p>
  </section>
  <section id="list-of-specifications-with-obsolete-webidl-constructs" class="level2">
  <h2>List of specifications with obsolete WebIDL constructs</h2>
  <p>=&gt; 0 specification found</p>
  </section>
  <section id="specifications-that-use-webidl-but-do-not-reference-the-webidl-spec" class="level2">
  <h2>Specifications that use WebIDL but do not reference the WebIDL spec</h2>
  <ul>
  <li><a href="https://w3c.github.io/staticrange/">Static Range</a></li>
  </ul>
  <p>=&gt; 1 specification found</p>
  </section>
  <section id="list-of-webidl-names-not-defined-in-the-specifications-crawled" class="level2">
  <h2>List of WebIDL names not defined in the specifications crawled</h2>
  <ul>
  <li><code>Navigator</code> used in <a href="https://wicg.github.io/mediasession/">Media Session Standard</a></li>
  <li><code>Node</code> used in <a href="https://w3c.github.io/staticrange/">Static Range</a></li>
  <li><code>Window</code> used in <a href="https://wicg.github.io/mediasession/">Media Session Standard</a></li>
  </ul>
  <p>=&gt; 3 WebIDL names found</p>
  <p>Some of them may be type errors in specs (e.g. &quot;int&quot; does not exist, &quot;Array&quot; cannot be used on its own, etc.) Also, please keep in mind that Reffy only knows about IDL terms defined in the specifications that were crawled <strong>and</strong> that do not have invalid IDL content.</p>
  </section>
  <section id="list-of-webidl-names-defined-in-more-than-one-spec" class="level2">
  <h2>List of WebIDL names defined in more than one spec</h2>
  <p>=&gt; 0 WebIDL name found</p>
  </section>
  <section id="missing-references-for-webidl-names" class="level2">
  <h2>Missing references for WebIDL names</h2>
  <p>=&gt; 0 missing reference for IDL definitions found in 0 specification</p>
  </section>
  <section id="missing-references-based-on-document-links" class="level2">
  <h2>Missing references based on document links</h2>
  <p>=&gt; 0 missing reference for links found in 0 specification</p>
  </section>
  <section id="reference-url-is-inconsistent-with-url-used-in-document-links" class="level2">
  <h2>Reference URL is inconsistent with URL used in document links</h2>
  <p>=&gt; 0 inconsistent reference for links found in 0 specification</p>
  </section>
  </body>
</html>